# SOME DESCRIPTIVE TITLE.
# Copyright (C) YEAR Red Hat, Inc.
# This file is distributed under the same license as the PACKAGE package.
#
# Translators:
# Bahadir Yagan <bahadir.yagan@mentorsystem.com>, 2004
# dcantrel <dcantrell@redhat.com>, 2011
# Dimitris Glezos <glezos@transifex.com>, 2011
# Ebubekir KARUL <ebubekirkarul@yandex.com>, 2013
# Egemen Metin Turan <mturan@aegee-ankara.org>, 2006
# Hasan Alp İNAN <hasanalpinan@gmail.com>, 2011
# Ismail ASCI <ismail.asci@gmail.com>, 2006
# Muhammet Kara <muhammetk@gmail.com>, 2011
# Onuralp SEZER <thunderbirdtr@gmail.com>, 2013
# Onur Gungor <onurgu@boun.edu.tr>, 2004, 2006
# Akın Ömeroğlu <akinomeroglu@gmail.com>, 2015. #zanata
# Berat Emre Nebioğlu <beratn@gmail.com>, 2016
# Emre Kerim Yıldız <ekyildizz@gmail.com>, 2016

msgid ""
msgstr ""
"Project-Id-Version: anaconda 24.13.5\n"
"Report-Msgid-Bugs-To: anaconda-devel-list@redhat.com\n"
"POT-Creation-Date: 2016-06-13 16:12-0700\n"
"PO-Revision-Date: 2015-06-07 05:27-0400\n"
"Last-Translator: Emre Kerim Yıldız <ekyildizz@gmail.com>\n"
"Language-Team: Turkish (http://www.transifex.com/projects/p/anaconda/"
"language/tr/)\n"
"Language: tr\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Plural-Forms: nplurals=2; plural=(n > 1);\n"
"X-Generator: Zanata 3.8.4\n"

#: anaconda.py:334
#, python-format
msgid ""
"%(product_name)s requires %(needed_ram)s MB of memory to install, but you "
"only have %(total_ram)s MB on this machine.\n"
msgstr ""
"%(product_name)s'un yüklenenebilmesi için %(needed_ram)s MB bellek gereklidir, Fakat "
"bu makina sadece %(total_ram)s MB belleğe sahiptir."

#: anaconda.py:336
#, python-format
msgid ""
"The %(product_name)s graphical installer requires %(needed_ram)s MB of "
"memory, but you only have %(total_ram)s MB\n"
"."
msgstr ""
"%(product_name)s'un grafik yükleyicisi %(needed_ram)s MB bellek gerektirmektedir, Fakat "
"%(total_ram)s MB belleğe sahipsin."

#: anaconda.py:339
msgid ""
"\n"
"Press <Enter> to reboot your system.\n"
msgstr ""
"\n"
"Sistemi yeniden başlatmak için <Enter>'a basın.\n"

#: anaconda.py:341
msgid "Not enough RAM"
msgstr "Yetersiz BELLEK"

#: anaconda.py:342
msgid ""
" Try the text mode installer by running:\n"
"\n"
"'/usr/bin/liveinst -T'\n"
"\n"
" from a root terminal."
msgstr ""
" Bir root terminalinde\n"
"\n"
"'/usr/bin/liveinst -T'\n"
"\n"
"komutunu çalıştırarak metin kipinde kurulumu deneyin."

#: anaconda.py:345
msgid " Starting text mode."
msgstr " Metin kipi başlatılıyor."

#: anaconda.py:381
msgid "The installation cannot continue and the system will be rebooted"
msgstr "Kurulum devam edemiyor. Bu yüzden sistem yeniden başlatılacak."

#: anaconda.py:382 pyanaconda/ui/tui/simpleline/base.py:571
msgid "Press ENTER to continue"
msgstr "Devam etmek için ENTER'a basın"

#: anaconda.py:505
msgid "DISPLAY variable not set. Starting text mode."
msgstr "DISPLAY değişkeni ayarlanmamış. Metin kipi başlatılıyor."

#: anaconda.py:513
msgid "Graphical installation is not available. Starting text mode."
msgstr "Grafik kipinde kurulum mevcut değil. Metin kipinde kuruluma geçiliyor."

#: anaconda.py:521
msgid ""
"Text mode provides a limited set of installation options. It does not offer "
"custom partitioning for full control over the disk layout. Would you like to "
"use VNC mode instead?"
msgstr ""
"Metin kipi sınırlı miktarda yükleme seçeneği sunmaktadır. Diski bölümlendirme "
"konusunda herhangi bir, kişisel ayar seçeneği sunmamaktadır. Disk bölümleme "
"işleminde tüm seçenekleri kullanabilmek için VNC modunu kullanın. "

#: anaconda.py:617
#, python-format
msgid "Please ssh install@%s to begin the install."
msgstr "Yüklemeyi başlatmak için ssh install@%s "

#: anaconda.py:619
msgid "Please ssh install@<host> to continue installation."
msgstr "Yüklemeye devam etmek için ssh install@<host> " 

#: anaconda.py:807
msgid "Unable to create PID file"
msgstr "PID dosyası oluşturulamıyor"

#: anaconda.py:808
#, python-format
msgid ""
"Anaconda is unable to create %s because the file already exists. Anaconda is "
"already running, or a previous instance of anaconda has crashed."
msgstr ""
"Anaconda dosya zaten var olduğu için %s isimli dosyayı oluşturamadı. Anaconda "
"zaten çalışıyor veya anaconda'nın bir önceki süreci durdu. "

#: pyanaconda/addons.py:87
#, python-format
msgid "Executing %s addon"
msgstr "%s eklentisi çalıştırılıyor."

#: pyanaconda/bootloader.py:226
msgid "/boot file system"
msgstr "/boot dosya sistemi"

#: pyanaconda/bootloader.py:383
#, python-format
msgid ""
"RAID sets that contain '%(desc)s' must have one of the following raid "
"levels: %(raid_level)s."
msgstr ""
"'%(desc)s' içeren RAID seviyeleri, aşağıdaki RAID seviyelerinden biri olmalıdır. "
"Seviyeler: %(raid_level)s."

#: pyanaconda/bootloader.py:391
#, python-format
msgid ""
"RAID sets that contain '%(desc)s' must have one of the following metadata "
"versions: %(metadata_versions)s."
msgstr ""
"'%(desc)s' içeren RAID seviyeleri , aşağıdaki versiyonlardan biri olmalı. "
"Versiyonlar: %(metadata_versions)s."

#: pyanaconda/bootloader.py:399
#, python-format
msgid ""
"RAID sets that contain '%(desc)s' must have one of the following device "
"types: %(types)s."
msgstr ""
"'%(desc)s' içeren RAID setleri aşağıdaki aygıtlardan biri olmalı. "
"Türler: %(types)s."

#: pyanaconda/bootloader.py:415
#, python-format
msgid "%(name)s must have one of the following disklabel types: %(types)s."
msgstr "%(name)s disk etiket türlerinden birine sahip olmalı. Türler: %(types)s."

#: pyanaconda/bootloader.py:427
#, python-format
msgid "%(desc)s cannot be of type %(type)s."
msgstr "%(desc)s türü %(type)s olamaz."

#: pyanaconda/bootloader.py:433
#, python-format
msgid "%(desc)s must be mounted on one of %(mountpoints)s."
msgstr "%(desc)s %(mountpoints)s bağlama noktalarından birine bağlanmalıdır."

#: pyanaconda/bootloader.py:445
#, python-format
msgid "%(desc)s must be between %(min)d and %(max)d MB in size"
msgstr "%(desc)s boyutu %(min)d ve %(max)d arasında olmalı"

#: pyanaconda/bootloader.py:451
#, python-format
msgid "%(desc)s must not be smaller than %(min)dMB."
msgstr "%(desc)s %(max)dMB dan büyük olmalı."

#: pyanaconda/bootloader.py:460
#, python-format
msgid "%(desc)s must not be larger than %(max)dMB."
msgstr "%(desc)s %(max)dMB dan büyük olamaz."

#: pyanaconda/bootloader.py:478
#, python-format
msgid "%(desc)s must be within the first %(max_end)s of the disk."
msgstr "%(desc)s diskin ilk %(max_end)s'ının içinde olmalı. "

#: pyanaconda/bootloader.py:488
#, python-format
msgid "%s must be on a primary partition."
msgstr "%s birincil bölüm üzerinde olmak zorunda."

#: pyanaconda/bootloader.py:624 pyanaconda/bootloader.py:724
#, python-format
msgid "%s cannot be on an encrypted block device."
msgstr "%s şifrelenmiş bir blok aygıtı üzerinde olamaz."

#: pyanaconda/bootloader.py:679
#, python-format
msgid "%s cannot be on an iSCSI disk"
msgstr "%s iSCSI diskin üzerinde olamaz."

#: pyanaconda/bootloader.py:683
#, python-format
msgid "%(desc)s cannot be of type %(type)s"
msgstr "%(desc)s %(type)s türünden olamaz."

#: pyanaconda/bootloader.py:1357
#, python-format
msgid ""
"boot loader stage2 device %(stage2dev)s is on a multi-disk array, but boot "
"loader stage1 device %(stage1dev)s is not. A drive failure in %(stage2dev)s "
"could render the system unbootable."
msgstr ""
"boot yükleyicisi stage2 aygıtı çok diskli bir %(stage2dev)s dizisinde, Fakat "
"boot yükleyicisi stage1 aygıtı %(stage1dev)s değil. %(stage2dev)s diskindeki "
"hata sistemi önyüklenemez hale getirebilir."

#: pyanaconda/bootloader.py:1363
#, python-format
msgid ""
"boot loader stage2 device %(stage2dev)s is on a multi-disk array, but boot "
"loader stage1 device %(stage1dev)s is not part of this array. The stage1 "
"boot loader will only be installed to a single drive."
msgstr ""
"boot yükleyicisi stage2 aygıtı çok diskli bir %(stage2dev)s dizisinde, Fakat "
"boot yükleyicisi stage1 aygıtı %(stage1dev)s dizisinin bir parçası değil. "
"Stage1 boot yükleyicisi yanlızca bir diske kurulabilir."

#: pyanaconda/bootloader.py:1658
#, python-format
msgid ""
"%(deviceName)s may not have enough space for grub2 to embed core.img when "
"using the %(fsType)s file system on %(deviceType)s"
msgstr ""
"%(deviceName)s grub2'nin core.img dosyasını gömmek için yeterli alana sahip değil."
"%(deviceType)s üzerinde %(fsType)s dosya sistemi kullanıyor"

#: pyanaconda/constants.py:73
msgid "Start VNC"
msgstr "VNC'yi başlat"

#: pyanaconda/constants.py:74
msgid "Use text mode"
msgstr "Metin modunu kullan"

#: pyanaconda/constants.py:138
msgid "The password is empty."
msgstr "Parola alanı boş."

#: pyanaconda/constants.py:139
msgid "The passwords do not match."
msgstr "Parolalar eşleşmiyor."

#: pyanaconda/constants.py:140
msgid "The passwords you entered were different.  Please try again."
msgstr "Girdiğiniz parolalar birbirinden farklı. Lütfen tekrar deneyin."

#: pyanaconda/constants.py:141
#, python-format
msgid "The password you have provided is weak. %s"
msgstr "Girdiğiniz parola zayıf. %s"

#: pyanaconda/constants.py:142
#, python-format
msgid "The password you have provided is weak: %s."
msgstr "Girdiğiniz parola zayıf: %s"

#: pyanaconda/constants.py:143
msgid "You have provided a weak password. Press Done again to use anyway."
msgstr "Zayıf bir parola girdiniz. Yine de kullanmak için Tamam'a basın."

#: pyanaconda/constants.py:144
#, python-format
msgid "You have provided a weak password: %s. Press Done again to use anyway."
msgstr "Zayıf bir parola girdiniz: %s. Yine de kullanmak için Tamam'a basın."

#: pyanaconda/constants.py:145
msgid ""
"The password you have provided contains non-ASCII characters. You may not be "
"able to switch between keyboard layouts to login. Press Done to continue."
msgstr ""
"Girdiğiniz parolada ASCII olmayan karakterler bulunmakta. Giriş yapmak için "
"farklı klavye örüntüleri arasında geçiş yapamayabilirsiniz. Devam etmek için "
"Tamam'a basın."

#: pyanaconda/constants.py:146
msgid "You will have to press Done twice to confirm it."
msgstr "Onaylamanız için Tamam tuşuna iki kere basmanız gerekecek."

#: pyanaconda/constants.py:148
msgid "Empty"
msgstr "Boş"

#: pyanaconda/constants.py:148 pyanaconda/ui/gui/spokes/lib/passphrase.py:149
#: pyanaconda/ui/gui/spokes/lib/passphrase.glade:180
msgid "Weak"
msgstr "Zayıf"

#: pyanaconda/constants.py:148 pyanaconda/ui/gui/spokes/lib/passphrase.py:152
msgid "Fair"
msgstr "Asgari"

#: pyanaconda/constants.py:148 pyanaconda/ui/gui/spokes/lib/passphrase.py:155
msgid "Good"
msgstr "İyi"

#: pyanaconda/constants.py:148 pyanaconda/ui/gui/spokes/lib/passphrase.py:158
msgid "Strong"
msgstr "Güçlü"

#: pyanaconda/constants.py:189
msgid "Probing storage..."
msgstr "Depolama inceleniyor..."

#: pyanaconda/constants.py:190
msgid "Downloading package metadata..."
msgstr "Paket meta verisi indiriliyor..."

#: pyanaconda/constants.py:191
msgid "Downloading group metadata..."
msgstr "Grup meta verisi indiriliyor..."

#: pyanaconda/constants.py:194
msgid "Anaconda Installer"
msgstr "Anaconda Yükleyici"

#: pyanaconda/errors.py:109
#, python-format
msgid ""
"The following errors occurred with your partitioning:\n"
"\n"
"%(errortxt)s\n"
"\n"
"The installation will now terminate."
msgstr ""
"Bölümlemenizde aşağıdaki hatalar oluştu:"
"\n"
"%(errortxt)s\n"
"\n"
"Yükleme iptal edilecek."

#: pyanaconda/errors.py:115
#, python-format
msgid "An error occurred while resizing the device %s."
msgstr "%s: Cihazı yeniden boyutlandırırken bir hata oluştu"

#: pyanaconda/errors.py:124
#, python-format
msgid ""
"There is a problem with your existing storage configuration: %(errortxt)s\n"
"\n"
"You must resolve this matter before the installation can proceed. There is a "
"shell available for use which you can access by pressing ctrl-alt-f1 and "
"then ctrl-b 2.\n"
"\n"
"Once you have resolved the issue you can retry the storage scan. If you do "
"not fix it you will have to exit the installer."
msgstr ""
"Varolan depolama alanı yapılandırmanızla ilgili bir sorun var: %(errortxt)s\n"
"\n"
"Kuruluma devam etmeden önce bu konuyu çözmeniz gerekir. Ctrl+alt+f1 ve "
"ctrl+b tuşlarına birlikte basarak kullanabileceğiniz bir kabuk \n."
"açabilirsiniz."
"\n"
"Sorunu çözdükten sonra, depolama alanı taramasını yeniden deneyebilirsiniz."
"Sorunu düzeltmezseniz yükleyiciden çıkmanız gerekecektir."

#: pyanaconda/errors.py:133
msgctxt "GUI|Storage Detailed Error Dialog"
msgid "_Exit Installer"
msgstr "Kurulumdan Çık"

#: pyanaconda/errors.py:134
msgctxt "GUI|Storage Detailed Error Dialog"
msgid "_Retry"
msgstr "Devam Et"

#: pyanaconda/errors.py:141
msgid ""
"An error has occurred - no valid devices were found on which to create new "
"file systems.  Please check your hardware for the cause of this problem."
msgstr ""
"Bir hata oluştu - Yeni dosya sistemini oluşturmak için geçerli bir cihaz "
"bulunamadı. Lütfen problemin nedeni olabilecek donanımınızı kontrol edin."

#: pyanaconda/errors.py:150
msgid ""
"There is an entry in your /etc/fstab file that contains an invalid or "
"incorrect file system type:\n"
"\n"
msgstr ""
"/etc/fstab dosyanızda geçersiz veya hatalı dosya sistemi türü "
"içeren bir giriş var:\n"
"\n"

#: pyanaconda/errors.py:156
#, python-format
msgid ""
"The ISO image %s has a size which is not a multiple of 2048 bytes.  This may "
"mean it was corrupted on transfer to this computer.\n"
"\n"
"It is recommended that you exit and abort your installation, but you can "
"choose to continue if you think this is in error. Would you like to continue "
"using this image?"
msgstr ""
"ISO görüntüsü %s, 2048 baytın katları olmayan bir boyuta sahip. Bu "
"bilgisayara aktarımın bozulduğu anlamına gelir. \n"
"\n"
"Kurulumdan çıkıp sonlandırmanız önerilir, ancak bunun yanlış olduğunu "
"düşünüyorsanız devam etmeyi seçebilirsiniz. Devam etmek ister misiniz?"
"Bu görüntüyü mü kullanıyorsunuz?"

#: pyanaconda/errors.py:170
msgid ""
"The installer has tried to mount the installation image, but cannot find it "
"on the hard drive.\n"
"\n"
"Should I try again to locate the image?"
msgstr ""
"Kurulum görüntüyü yüklemeye çalıştı ancak disk üzerinde "
"bulamıyor.\n"
"\n"
"Görüntüyü bulmak için tekrar denemelimiyim?"

#: pyanaconda/errors.py:180
#, python-format
msgid "An error occurred mounting the source device %s. Retry?"
msgstr "Kaynak %s cihazını bağlarken bir hata oluştu. Tekrar deneyin?"

#: pyanaconda/errors.py:188
#, python-format
msgid ""
"An error occurred unmounting the disc.  Please make sure you're not "
"accessing %s from the shell on tty2 and then click OK to retry."
msgstr ""
"Diskin bağını kaldırırken bir hata oluştu. %s diskine tty2 kabuğundan "
"erişmediğinize emin olun ve tekrar denemek için Tamam'ı tıklayın"

#: pyanaconda/errors.py:196
#, python-format
msgid ""
"The group '%s' is required for this installation. This group does not exist. "
"This is a fatal error and installation will be aborted."
msgstr ""
"Grup '%s' kurulum için gereklidir. Grup bulunamadı. "
"Bu ölümcül bir hata ve kurulum iptal edilecek."

#: pyanaconda/errors.py:202
#, python-format
msgid ""
"You have specified that the group '%s' should be installed.  This group does "
"not exist.  Would you like to ignore this group and continue with "
"installation?"
msgstr ""
"'%s' grubu'nun kurulması gerektiğini belirttiniz. Grup bulunamadı.  "
"Bu grubu yok yasayarak kuruluma devam etmek "
"istermisiniz? "

#: pyanaconda/errors.py:207
#, python-format
msgid ""
"You have specified that the group '%s' should be excluded from "
"installation.  This group does not exist.  Would you like to ignore this "
"group and continue with installation?"
msgstr ""
"'%s' grubu'nun kurulumdan hariç tutulması gerektiğini belirttiniz. Bu grup "
"bulunamadı. Bu grubu görmezden gelip kuruluma devam etmek "
"istermisiniz?"

#: pyanaconda/errors.py:219
#, python-format
msgid ""
"The package '%s' is required for this installation. This package does not "
"exist. This is a fatal error and installation will be aborted."
msgstr ""
"'%s' paketi kurulum için gereklidir. Paket bulunamadı. Bu ölümcül "
"bir hatadır bu sebeple kurulum iptal edilecek."

#: pyanaconda/errors.py:226
#, python-format
msgid ""
"You have specified that the package '%s' should be installed.  This package "
"does not exist.  Would you like to ignore this package and continue with "
"installation?"
msgstr ""
"'%s' paketi'nin kurulması gerektiğini belirttiniz. Paket bulunamadı. "
"Bu paketi yok sayarak kuruluma devam etmek "
"istermisiniz?"

#: pyanaconda/errors.py:237
#, python-format
msgid ""
"There was an error running the kickstart script at line %(lineno)s.  This is "
"a fatal error and installation will be aborted.  The details of this error "
"are:\n"
"\n"
"%(details)s"
msgstr ""
"%(lineno)s satırında kickstart komut dosyasını çalıştırırken bir hata oluştu. "
"Bu ölümcül bir hata olduğu için kurulum iptal edilecek. Hata'nın "
"ayrıntıları:\n"
"\n"
"%(details)s"

#: pyanaconda/errors.py:245
msgid ""
"The following error occurred while installing.  This is a fatal error and "
"installation will be aborted."
msgstr ""
"Yükleme sırasında ekteki hata oluştu. Bu ölümcül bir hata olduğu için yükleme "
"işlemi iptal edilecek."

#: pyanaconda/errors.py:253
msgid ""
"The following software marked for installation has errors.  This is likely "
"caused by an error with\n"
"your installation source."
msgstr ""
"Kurulum için aşağıda işaretlenmiş yazılım hataları var. "
"Büyük olasılıkla yükleme kaynağınızdaki \n"
"bir hatadan kaynaklanıyor."

#: pyanaconda/errors.py:261
msgid ""
"The following error occurred while installing the boot loader. The system "
"will not be bootable. Would you like to ignore this and continue with "
"installation?"
msgstr ""
"Boot yükleyicisi kurulurken aşağıdaki hatalar oluştu. "
"Sistem önyüklenebilir olmayacak. Bu hatayı görmezden gelip kuruluma "
"devam etmek istiyormusunuz?"

#: pyanaconda/errors.py:273
#, python-format
msgid "Unable to encrypt password: unsupported algorithm %s"
msgstr "Parola şifrelenemiyor: desteklenmeyen algoritma %s"

#: pyanaconda/errors.py:280
#, python-format
msgid ""
"Installation was stopped due to an error installing the boot loader. The "
"exact error message is:\n"
"\n"
"%s\n"
"\n"
"The installer will now terminate."
msgstr ""
"Boot yükleyicisi kurulamadığından kurulum durdu. "
"Hata mesajı:\n"
"\n"
"%s\n"
"\n"
"Kurulumda çıkılıyor."

#: pyanaconda/exception.py:106
#, python-format
msgid ""
"The installation was stopped due to what seems to be a problem with your "
"hardware. The exact error message is:\n"
"\n"
"%s.\n"
"\n"
" The installer will now terminate."
msgstr ""
"Donanımınızla ilgili bir hatadan dolayı kurulum durdu. "
"Hata mesajı:\n"
"\n"
"%s.\n"
"\n"
" Kurulumda çıkılıyor."

#: pyanaconda/exception.py:110
msgid "Hardware error occurred"
msgstr "Donanım hatası oluştu"

#: pyanaconda/exception.py:167
#, python-format
msgid ""
"\n"
"The installation was stopped due to incomplete spokes detected while running "
"in non-interactive cmdline mode. Since there cannot be any questions in "
"cmdline mode, edit your kickstart file and retry installation.\n"
"The exact error message is: \n"
"\n"
"%s.\n"
"\n"
"The installer will now terminate."
msgstr ""
"\n"
"Etkileşimsiz cmdline modunda koşulan komutlarda hata olması nedeniyle "
"kurulum durdu. Bu mod'da kuruluma devam edemezsiniz. Kickstart  "
"dosyanızı düzenleyin ve kurulumu cmdline modunda tekrar başlatın.\n"
"Hata mesajı: \n"
"\n"
"%s.\n"
"\n"
"Kurulumdan çıkılıyor."

#: pyanaconda/exception.py:175
#, python-format
msgid ""
"\n"
"Running in cmdline mode, no interactive debugging allowed.\n"
"The exact error message is: \n"
"\n"
"%s.\n"
"\n"
"The installer will now terminate."
msgstr ""
"\n"
"Cmdline modunda çalışıyor. Etkileşimli hata ayıklama yapılmasına izin yok.\n"
"Hata mesajı: \n"
"\n"
"%s.\n"
"\n"
"Kurulumdan çıkılıyor."

#: pyanaconda/install.py:75
msgid "Configuring installed system"
msgstr "Yüklenen sistem ayarlanıyor"

#: pyanaconda/install.py:88
msgid "Writing network configuration"
msgstr "Ağ ayarları yazılıyor"

#: pyanaconda/install.py:92
msgid "Creating users"
msgstr "Kullanıcılar oluşturuluyor"

#: pyanaconda/install.py:99
msgid "Configuring addons"
msgstr "Eklentiler ayarlanıyor"

#: pyanaconda/install.py:102
msgid "Generating initramfs"
msgstr "İnitramfs oluşturuluyor"

#: pyanaconda/install.py:113
#, python-format
msgid "Joining realm: %s"
msgstr "%s alanına katılınıyor"

#: pyanaconda/install.py:116
msgid "Running post-installation scripts"
msgstr "Yükleme sonrası betikleri çalıştırılıyor"

#: pyanaconda/install.py:168
#, python-format
msgid "Waiting for %s threads to finish"
msgstr "%s işlemlerinin tamamlanması bekleniyor"

#: pyanaconda/install.py:175
msgid "Setting up the installation environment"
msgstr "Yükleme ortamı oluşturuluyor"

#: pyanaconda/install.py:199 pyanaconda/kickstart.py:2177
msgid "Running pre-installation scripts"
msgstr "Kurulum öncesi betikleri çalıştırılıyor"

#: pyanaconda/install.py:207
msgid "Discovering realm to join"
msgstr "Katılınacak alan keşfediliyor"

#: pyanaconda/install.py:242
msgid "Installing boot loader"
msgstr "Boot yükleyicisi yükleniyor"

#: pyanaconda/install.py:245 pyanaconda/packaging/livepayload.py:182
#: pyanaconda/packaging/dnfpayload.py:817
msgid "Performing post-installation setup tasks"
msgstr "Yükleme sonrası ayar görevleri çalıştırılıyor"

#: pyanaconda/iutil.py:859
msgid "No host url"
msgstr "Sunucu URL'si yok"

#: pyanaconda/iutil.py:880
msgid "malformed URL, cannot parse it."
msgstr "Hatalı URL, çözümlenemiyor."

#: pyanaconda/iutil.py:896
msgid "URL has no host component"
msgstr "URL sunucu bileşeni içermiyor"

#: pyanaconda/kickstart.py:160
#, python-format
msgid "Escrow certificate %s requires the network."
msgstr "Geçici sertifika %s ağa ihtiyaç duyuyor."

#: pyanaconda/kickstart.py:168
#, python-format
msgid ""
"SSL error while downloading the escrow certificate:\n"
"\n"
"%s"
msgstr ""
"Geçici sertifika indirilirken SSL hatası:\n"
"\n"
"%s"

#: pyanaconda/kickstart.py:171
#, python-format
msgid ""
"The following error was encountered while downloading the escrow "
"certificate:\n"
"\n"
"%s"
msgstr ""
"Karşıdan indirilirken aşağıdaki hata ile karşılaşıldı. "
"Sertifika:\n"
"\n"
"%s"

#: pyanaconda/kickstart.py:282
#, python-format
msgid "%s is missing. Cannot setup authentication."
msgstr "%s bulunamıyor. Yetkilendirme ayarlanamıyor."

#: pyanaconda/kickstart.py:307
#, python-format
msgid "autopart fstype of %s is invalid."
msgstr "otomatik bölümleme dosya sistemi tipi %s geçersiz."

#: pyanaconda/kickstart.py:324
#, python-format
msgid "Settings default fstype to %s failed."
msgstr "Varsayılan ayarlardaki dosya sistemi tipi %s geçersiz"

#: pyanaconda/kickstart.py:355
msgid "GRUB2 does not support installation to a partition."
msgstr "GRUB2'nin bir bölüme yüklenmesi desteklenmiyor."

#: pyanaconda/kickstart.py:415
#, python-format
msgid "More than one match found for given boot drive \"%s\"."
msgstr "Belirli bir önyükleme sürücüsü için birden fazla eşleşme bulundu \"%s\"."

#: pyanaconda/kickstart.py:418
#, python-format
msgid "Requested boot drive \"%s\" doesn't exist or cannot be used."
msgstr "İstenen önyükleme bölümü \"%s\" mevcut değil."

#: pyanaconda/kickstart.py:460
#, python-format
msgid ""
"Btrfs partition \"%(device)s\" has a format of \"%(format)s\", but should "
"have a format of \"btrfs\"."
msgstr ""

#: pyanaconda/kickstart.py:465
#, python-format
msgid "Tried to use undefined partition \"%s\" in Btrfs volume specification."
msgstr ""

#: pyanaconda/kickstart.py:478
msgid ""
"Btrfs volume defined without any member devices.  Either specify member "
"devices or use --useexisting."
msgstr ""

#: pyanaconda/kickstart.py:487 pyanaconda/kickstart.py:864
#: pyanaconda/kickstart.py:1477
#, python-format
msgid "The mount point \"%s\" is not valid.  It must start with a /."
msgstr "Bağlantı noktası %s geçerli değil. / işaretiyle başlaması gerekiyor."

#: pyanaconda/kickstart.py:502
#, python-format
msgid "Btrfs volume \"%s\" specified with --useexisting does not exist."
msgstr ""

#: pyanaconda/kickstart.py:589
#, python-format
msgid ""
"Disklabel \"%s\" given in clearpart command is not supported on this "
"platform."
msgstr ""

#: pyanaconda/kickstart.py:601
#, python-format
msgid "Disk \"%s\" given in clearpart command does not exist."
msgstr ""

#: pyanaconda/kickstart.py:614
#, python-format
msgid "Device \"%s\" given in clearpart device list does not exist."
msgstr ""

#: pyanaconda/kickstart.py:641
#, python-format
msgid "NIC \"%s\" given in fcoe command does not exist."
msgstr ""

#: pyanaconda/kickstart.py:692
#, python-format
msgid "%s is missing. Cannot setup firewall."
msgstr "%s yok. Güvenlik duvarı ayarlanamıyor."

#: pyanaconda/kickstart.py:743 pyanaconda/kickstart.py:754
#, python-format
msgid "Disk \"%s\" given in ignoredisk command does not exist."
msgstr ""

#: pyanaconda/kickstart.py:767
#, python-format
msgid ""
"Network interface \"%(nic)s\" required by iSCSI \"%(iscsiTarget)s\" target "
"is not up."
msgstr ""

#: pyanaconda/kickstart.py:777
msgid ""
"iscsi --iface must be specified (binding used) either for all targets or for "
"none"
msgstr ""

#: pyanaconda/kickstart.py:870
#, python-format
msgid ""
"No volume group exists with the name \"%s\".  Specify volume groups before "
"logical volumes."
msgstr ""

#: pyanaconda/kickstart.py:877
msgid "Cache PVs must belong to the same VG as the cached LV"
msgstr ""

#: pyanaconda/kickstart.py:884
#, python-format
msgid ""
"No thin pool exists with the name \"%s\". Specify thin pools before thin "
"volumes."
msgstr ""

#: pyanaconda/kickstart.py:892
msgid "logvol --noformat must also use the --name= option."
msgstr ""

#: pyanaconda/kickstart.py:897 pyanaconda/kickstart.py:959
#, python-format
msgid "Logical volume \"%s\" given in logvol command does not exist."
msgstr ""

#: pyanaconda/kickstart.py:908 pyanaconda/kickstart.py:917
#: pyanaconda/kickstart.py:969 pyanaconda/kickstart.py:1257
#: pyanaconda/kickstart.py:1266 pyanaconda/kickstart.py:1349
#, python-format
msgid "Target size \"%(size)s\" for device \"%(device)s\" is invalid."
msgstr ""

#: pyanaconda/kickstart.py:931
#, python-format
msgid ""
"Logical volume name \"%(logvol)s\" is already in use in volume group "
"\"%(volgroup)s\"."
msgstr ""

#: pyanaconda/kickstart.py:936
#, python-format
msgid ""
"Logical volume size \"%(logvolSize)s\" must be larger than the volume group "
"extent size of \"%(extentSize)s\"."
msgstr ""

#: pyanaconda/kickstart.py:948 pyanaconda/kickstart.py:1285
#: pyanaconda/kickstart.py:1530
#, python-format
msgid "The \"%s\" file system type is not supported."
msgstr ""

#: pyanaconda/kickstart.py:1123 pyanaconda/kickstart.py:1127
#, python-format
msgid "The %s kickstart command is not currently supported."
msgstr "%s kickstart komutu şu anda desteklenmiyor."

#: pyanaconda/kickstart.py:1154
#, python-format
msgid "No disk found for specified BIOS disk \"%s\"."
msgstr ""

#: pyanaconda/kickstart.py:1189
#, python-format
msgid "RAID partition \"%s\" is defined multiple times."
msgstr ""

#: pyanaconda/kickstart.py:1200 pyanaconda/kickstart.py:1452
#, python-format
msgid "PV partition \"%s\" is defined multiple times."
msgstr ""

#: pyanaconda/kickstart.py:1211 pyanaconda/kickstart.py:1462
#, python-format
msgid "Btrfs partition \"%s\" is defined multiple times."
msgstr ""

#: pyanaconda/kickstart.py:1234
#, python-format
msgid "The size \"%s\" is invalid."
msgstr "%s büyüklüğü geçerli değil."

#: pyanaconda/kickstart.py:1241
msgid "part --noformat must also use the --onpart option."
msgstr ""

#: pyanaconda/kickstart.py:1246 pyanaconda/kickstart.py:1340
#, python-format
msgid "Partition \"%s\" given in part command does not exist."
msgstr ""

#: pyanaconda/kickstart.py:1300 pyanaconda/kickstart.py:1314
#, python-format
msgid "Disk \"%s\" given in part command does not exist."
msgstr ""

#: pyanaconda/kickstart.py:1303
#, python-format
msgid "Cannot install to unpartitionable device \"%s\"."
msgstr "Biçimlendirilmeyen %s cihazına yükleme yapılamıyor."

#: pyanaconda/kickstart.py:1310
#, python-format
msgid "Disk \"%s\" in part command is not partitioned."
msgstr ""

#: pyanaconda/kickstart.py:1323
#, python-format
msgid "The maximum size \"%s\" is invalid."
msgstr ""

#: pyanaconda/kickstart.py:1484
msgid "raid --noformat must also use the --device option."
msgstr ""

#: pyanaconda/kickstart.py:1489
#, python-format
msgid "RAID device  \"%s\" given in raid command does not exist."
msgstr ""

#: pyanaconda/kickstart.py:1512
#, python-format
msgid ""
"RAID device \"%(device)s\" has a format of \"%(format)s\", but should have a "
"format of \"mdmember\"."
msgstr ""

#: pyanaconda/kickstart.py:1517
#, python-format
msgid "Tried to use undefined partition \"%s\" in RAID specification."
msgstr ""

#: pyanaconda/kickstart.py:1546
#, python-format
msgid "RAID volume \"%s\" specified with --useexisting does not exist."
msgstr ""

#: pyanaconda/kickstart.py:1555
#, python-format
msgid "The RAID volume name \"%s\" is already in use."
msgstr ""

#: pyanaconda/kickstart.py:1805
#, python-format
msgid ""
"Physical volume \"%(device)s\" has a format of \"%(format)s\", but should "
"have a format of \"lvmpv\"."
msgstr ""

#: pyanaconda/kickstart.py:1810
#, python-format
msgid "Tried to use undefined partition \"%s\" in Volume Group specification"
msgstr ""

#: pyanaconda/kickstart.py:1816
#, python-format
msgid ""
"Volume group \"%s\" defined without any physical volumes.  Either specify "
"physical volumes or use --useexisting."
msgstr ""

#: pyanaconda/kickstart.py:1826
#, python-format
msgid ""
"Volume group given physical extent size of \"%(extentSize)s\", but must be "
"one of:\n"
"%(validExtentSizes)s."
msgstr ""

#: pyanaconda/kickstart.py:1833
msgid ""
"volgroup --noformat and volgroup --useexisting must also use the --name= "
"option."
msgstr ""

#: pyanaconda/kickstart.py:1838
#, python-format
msgid "Volume group \"%s\" given in volgroup command does not exist."
msgstr ""

#: pyanaconda/kickstart.py:1841
#, python-format
msgid "The volume group name \"%s\" is already in use."
msgstr ""

#: pyanaconda/kickstart.py:1897
msgid ""
"The upgrade kickstart command is no longer supported. Upgrade functionality "
"is provided through fedup."
msgstr ""
"Kickstart güncelle komutu artık desteklenmiyor. Güncelleme özelliği fedup "
"ile sağlanmaktadır."

#: pyanaconda/network.py:119
msgid "Host name cannot be None or an empty string."
msgstr "Sunucu ismi None ya da boş olamaz."

#: pyanaconda/network.py:122
msgid "Host name must be 255 or fewer characters in length."
msgstr "Sunucu ismi 255 ya da daha az karakter uzunluğunda olmalıdır."

#: pyanaconda/network.py:125
msgid ""
"Host names can only contain the characters 'a-z', 'A-Z', '0-9', '-', or '.', "
"parts between periods must contain something and cannot start or end with "
"'-'."
msgstr ""

#: pyanaconda/network.py:1403 pyanaconda/ui/gui/spokes/custom.py:579
#: pyanaconda/ui/gui/spokes/network.py:273
#: pyanaconda/ui/gui/spokes/lib/accordion.py:70
msgid "Unknown"
msgstr "Bilinmeyen"

#: pyanaconda/network.py:1407
msgid "Connecting..."
msgstr "Bağlanıyor..."

#: pyanaconda/network.py:1409
msgid "Disconnecting..."
msgstr "Bağlantı kesiliyor..."

#: pyanaconda/network.py:1429
#, python-format
msgid "Wired (%(interface_name)s) connected"
msgstr "Kablolu arayüz (%(interface_name)s) bağlandı"

#: pyanaconda/network.py:1432
#, python-format
msgid "Wireless connected to %(access_point)s"
msgstr "Kablosuz bağlantı  %(access_point)s ağına bağlı"

#: pyanaconda/network.py:1435
#, python-format
msgid "Bond %(interface_name)s (%(list_of_slaves)s) connected"
msgstr ""

#: pyanaconda/network.py:1439
#, python-format
msgid "Team%(interface_name)s (%(list_of_slaves)s) connected"
msgstr ""

#: pyanaconda/network.py:1443
#, python-format
msgid "Bridge%(interface_name)s (%(list_of_slaves)s) connected"
msgstr ""

#: pyanaconda/network.py:1449
#, python-format
msgid "VLAN %(interface_name)s (%(parent_device)s, ID %(vlanid)s) connected"
msgstr ""

#: pyanaconda/network.py:1466
#, python-format
msgid "Connected: %(list_of_interface_names)s"
msgstr ""

#: pyanaconda/network.py:1469
msgid "Not connected"
msgstr "Bağlı değil"

#: pyanaconda/network.py:1472 pyanaconda/ui/gui/spokes/network.glade:2052
msgid "No network devices available"
msgstr "Hiç bir ağ cihazı uygun değil"

#: pyanaconda/product.py:53
#, python-format
msgid "%(productName)s %(productVersion)s INSTALLATION"
msgstr "%(productName)s %(productVersion)s KURULUMU"

#: pyanaconda/product.py:57
#, python-format
msgid "New %(name)s %(version)s Installation"
msgstr ""

#: pyanaconda/pwpolicy.py:114
#, python-format
msgid "policy name required for %s"
msgstr ""

#: pyanaconda/pwpolicy.py:123
#, python-format
msgid "A %(command)s with the name %(policyName)s has already been defined."
msgstr ""

#: pyanaconda/rescue.py:67
#, python-format
msgid "Run %s to unmount the system when you are finished."
msgstr "Bitirdiğiniz zaman %s 'i çalıştırarak sistemi ayırınız."

#: pyanaconda/rescue.py:70
msgid "When finished, please exit from the shell and your system will reboot."
msgstr ""

#: pyanaconda/rescue.py:78
msgid "Unable to find /bin/bash to execute!  Not starting shell."
msgstr ""

#: pyanaconda/rescue.py:120
msgid "Rescue"
msgstr "Kurtar"

#: pyanaconda/rescue.py:132
msgid "Continue"
msgstr "Devam Et"

#: pyanaconda/rescue.py:132
msgid "Read-only mount"
msgstr ""

#: pyanaconda/rescue.py:132
msgid "Skip to shell"
msgstr ""

#: pyanaconda/rescue.py:146
msgid "Please make a selection from the above:  "
msgstr ""

#: pyanaconda/rescue.py:151
#, python-format
msgid ""
"The rescue environment will now attempt to find your Linux installation and "
"mount it under the directory : %s.  You can then make any changes required "
"to your system.  Choose '1' to proceed with this step.\n"
"You can choose to mount your file systems read-only instead of read-write by "
"choosing '2'.\n"
"If for some reason this process does not work choose '3' to skip directly to "
"a shell.\n"
"\n"
msgstr ""

#: pyanaconda/rescue.py:265
msgid "Root Selection"
msgstr ""

#: pyanaconda/rescue.py:282
msgid "The following installations were discovered on your system.\n"
msgstr ""

#: pyanaconda/rescue.py:294
#, python-format
msgid ""
"Please make your selection from the above list.\n"
"Press '%(continue)s' to continue after you have made your selection.  "
msgstr ""

#. TRANSLATORS:'c' to continue
#: pyanaconda/rescue.py:297
msgctxt "TUI|Root Selection"
msgid "c"
msgstr ""

#: pyanaconda/rescue.py:327
msgid "Rescue Mount"
msgstr ""

#: pyanaconda/rescue.py:346
#, python-format
msgid ""
"Your system has been mounted under %(mountpoint)s.\n"
"\n"
"If you would like to make your system the root environment, run the "
"command:\n"
"\n"
"\tchroot %(mountpoint)s\n"
msgstr ""

#: pyanaconda/rescue.py:385
#, python-format
msgid "Run %s to unmount the system when you are finished.\n"
msgstr ""

#: pyanaconda/rescue.py:387
#, python-format
msgid ""
"An error occurred trying to mount some or all of your system.  Some of it "
"may be mounted under %s\n"
"\n"
msgstr ""

#: pyanaconda/rescue.py:394
msgid "You don't have any Linux partitions.  Rebooting.\n"
msgstr "Herhangi bir Linux bölümü bulunamadı. Yeniden başlatılıyor.\n"

#: pyanaconda/rescue.py:401
msgid "The system will reboot automatically when you exit from the shell.\n"
msgstr ""

#: pyanaconda/rescue.py:405
#, python-format
msgid "You don't have any Linux partitions. %s\n"
msgstr ""

#: pyanaconda/rescue.py:415
#, python-format
msgid "Your system is mounted under the %s directory."
msgstr "Dosya sisteminiz %s altına bağlandı."

#: pyanaconda/rescue.py:436
msgid "Please press <Enter> to get a shell. "
msgstr ""

#: pyanaconda/storage_utils.py:52 pyanaconda/storage_utils.py:84
#: pyanaconda/ui/gui/spokes/custom.glade:31
msgid "LVM"
msgstr "LVM"

#: pyanaconda/storage_utils.py:53 pyanaconda/storage_utils.py:85
#: pyanaconda/ui/gui/spokes/custom.glade:39
msgid "LVM Thin Provisioning"
msgstr "LVM Thin Provisioning"

#: pyanaconda/storage_utils.py:54 pyanaconda/ui/gui/spokes/custom.glade:35
msgid "RAID"
msgstr "RAID"

#: pyanaconda/storage_utils.py:55 pyanaconda/storage_utils.py:82
#: pyanaconda/ui/gui/spokes/custom.glade:23
msgid "Standard Partition"
msgstr "Standart Alan"

#: pyanaconda/storage_utils.py:56 pyanaconda/storage_utils.py:83
#: pyanaconda/ui/gui/spokes/custom.glade:27
msgid "Btrfs"
msgstr "Btrfs"

#: pyanaconda/storage_utils.py:57 pyanaconda/ui/gui/spokes/lib/resize.glade:152
msgid "Disk"
msgstr "Disk"

#: pyanaconda/storage_utils.py:68
msgid ""
"The 'swap' area on your computer is used by the operating\n"
"system when running low on memory."
msgstr ""
"'Swap' alanı işletim sisteminiz tarafından\n"
"belleğiniz az olduğunda kullanılır."

#: pyanaconda/storage_utils.py:70
msgid ""
"The 'boot' area on your computer is where files needed\n"
"to start the operating system are stored."
msgstr ""

#: pyanaconda/storage_utils.py:72
msgid ""
"The 'root' area on your computer is where core system\n"
"files and applications are stored."
msgstr ""

#: pyanaconda/storage_utils.py:74
msgid ""
"The 'home' area on your computer is where all your personal\n"
"data is stored."
msgstr ""

#: pyanaconda/storage_utils.py:76
msgid ""
"The BIOS boot partition is required to enable booting\n"
"from GPT-partitioned disks on BIOS hardware."
msgstr ""

#: pyanaconda/storage_utils.py:78
msgid ""
"The PReP boot partition is required as part of the\n"
"boot loader configuration on some PPC platforms."
msgstr ""

#: pyanaconda/storage_utils.py:186
#, python-format
msgid ""
"Your root partition is less than 250 megabytes which is usually too small to "
"install %s."
msgstr ""

#: pyanaconda/storage_utils.py:191
#, python-format
msgid ""
"You have not defined a root partition (/), which is required for "
"installation of %s to continue."
msgstr ""

#: pyanaconda/storage_utils.py:205
msgid ""
"This platform requires /boot on a dedicated partition or logical volume.  If "
"you do not want a /boot volume, you must place / on a dedicated non-LVM "
"partition."
msgstr ""

#: pyanaconda/storage_utils.py:215
#, python-format
msgid ""
"Your %(mount)s partition is less than %(size)s which is lower than "
"recommended for a normal %(productName)s install."
msgstr ""

#: pyanaconda/storage_utils.py:227
#, python-format
msgid ""
"Your %(mount)s partition is too small for %(format)s formatting (allowable "
"size is %(minSize)s to %(maxSize)s)"
msgstr ""

#: pyanaconda/storage_utils.py:233
#, python-format
msgid ""
"Your %(mount)s partition is too large for %(format)s formatting (allowable "
"size is %(minSize)s to %(maxSize)s)"
msgstr ""

#: pyanaconda/storage_utils.py:242
msgid "No valid boot loader target device found. See below for details."
msgstr ""

#: pyanaconda/storage_utils.py:254
msgid "You have not created a bootable partition."
msgstr ""

#: pyanaconda/storage_utils.py:276
msgid ""
"Your BIOS-based system needs a special partition to boot from a GPT disk "
"label. To continue, please create a 1MiB 'biosboot' type partition."
msgstr ""

#: pyanaconda/storage_utils.py:287
#, python-format
msgid ""
"You have not specified a swap partition.  %(requiredMem)s of memory is "
"required to continue installation without a swap partition, but you only "
"have %(installedMem)s."
msgstr ""

#: pyanaconda/storage_utils.py:294
msgid ""
"You have not specified a swap partition.  Although not strictly required in "
"all cases, it will significantly improve performance for most installations."
msgstr ""

#: pyanaconda/storage_utils.py:301
msgid ""
"At least one of your swap devices does not have a UUID, which is common in "
"swap space created using older versions of mkswap. These devices will be "
"referred to by device path in /etc/fstab, which is not ideal since device "
"paths can change under a variety of circumstances. "
msgstr ""

#: pyanaconda/storage_utils.py:312
#, python-format
msgid ""
"This mount point is invalid.  The %s directory must be on the / file system."
msgstr ""

#: pyanaconda/storage_utils.py:317
#, python-format
msgid "The mount point %s must be on a linux file system."
msgstr ""

#: pyanaconda/storage_utils.py:345
#, python-format
msgid ""
"Encryption requested for LUKS device %s but no encryption key specified for "
"this device."
msgstr ""

#: pyanaconda/vnc.py:74
#, python-format
msgid "%(productName)s %(productVersion)s installation"
msgstr "%(productName)s %(productVersion)s kurulumu"

#: pyanaconda/vnc.py:138
#, python-format
msgid "%(productName)s %(productVersion)s installation on host %(name)s"
msgstr "%(name)s konağına %(productName)s %(productVersion)s kurulumu"

#: pyanaconda/vnc.py:157
#, python-format
msgid "Attempting to connect to vnc client on host %s..."
msgstr "Makine %s'te bulunan vnc istemcisi bağlantısı..."

#: pyanaconda/vnc.py:171
msgid "Connected!"
msgstr "Bağlandı!"

#: pyanaconda/vnc.py:174
msgid "Will try to connect again in 15 seconds..."
msgstr "15 saniye içinde tekrar bağlanmayı deneyecek..."

#: pyanaconda/vnc.py:181
#, python-format
msgid "Giving up attempting to connect after %d try!\n"
msgid_plural "Giving up attempting to connect after %d tries!\n"
msgstr[0] ""

#: pyanaconda/vnc.py:189
msgid "Attempting to start vncconfig"
msgstr "vncconfig başlatmayı deniyor"

#: pyanaconda/vnc.py:202
#, python-format
msgid "Please manually connect your vnc client to %s to begin the install."
msgstr "Kuruluma başlamak için lütfen %s'e vnc istemcisi ile bağlanın."

#: pyanaconda/vnc.py:204
#, python-format
msgid ""
"Please manually connect your vnc client to <IP ADDRESS>:%s to begin the "
"install. Switch to the shell (Ctrl-B 2) and run 'ip addr' to find the <IP "
"ADDRESS>."
msgstr ""

#: pyanaconda/vnc.py:209
msgid "Starting VNC..."
msgstr "VNC Başlatılıyor..."

#: pyanaconda/vnc.py:246
msgid "The VNC server is now running."
msgstr "VNC sunucusu şimdi çalışıyor."

#: pyanaconda/vnc.py:250
msgid ""
"\n"
"\n"
"You chose to connect to a listening vncviewer. \n"
"This does not require a password to be set.  If you \n"
"set a password, it will be used in case the connection \n"
"to the vncviewer is unsuccessful\n"
"\n"
msgstr ""

#: pyanaconda/vnc.py:255
msgid ""
"\n"
"\n"
"WARNING!!! VNC server running with NO PASSWORD!\n"
"You can use the vncpassword=<password> boot option\n"
"if you would like to secure the server.\n"
"\n"
msgstr ""
"\n"
"\n"
"UYARI!!! VNC sunucusu PAROLASIZ çalışıyor!\n"
"Eğer sunucuyu daha güvenli hale getirmek isterseniz\n"
" vncpassword=<parola> açılış seçeneğini kullanabilirsiniz.\n"
"\n"

#: pyanaconda/vnc.py:259
msgid ""
"\n"
"\n"
"You chose to execute vnc with a password. \n"
"\n"
msgstr ""
"\n"
"\n"
"VNC'yi bir parola ile çalıştırmayı seçtiniz. \n"
"\n"

#: pyanaconda/vnc.py:261
msgid ""
"\n"
"\n"
"Unknown Error.  Aborting. \n"
"\n"
msgstr ""
"\n"
"\n"
"Bilinmeyen Hata.  Durduruluyor. \n"
"\n"

#: pyanaconda/vnc.py:283
msgid ""
"VNC password must be six to eight characters long.\n"
"Please enter a new one, or leave blank for no password."
msgstr ""

#: pyanaconda/packaging/__init__.py:1179
msgid "Failed to set up installation source"
msgstr "Kurulum kaynağı oluşturulamadı"

#: pyanaconda/packaging/__init__.py:1180
msgid "Error downloading package metadata"
msgstr "Paket bilgisi indirilirken hata oluştu"

#: pyanaconda/packaging/__init__.py:1181
msgid "No installation source available"
msgstr "Kurulum kaynağı yok"

#: pyanaconda/packaging/livepayload.py:106
#: pyanaconda/packaging/livepayload.py:127
msgid "Installing software"
msgstr "Yüklendi"

#: pyanaconda/packaging/livepayload.py:237
#: pyanaconda/packaging/livepayload.py:246
#, python-format
msgid "Downloading %(url)s (%(pct)d%%)"
msgstr "İndiriliyor %(url)s (%(pct)d%%)"

#: pyanaconda/packaging/livepayload.py:390
msgid "Checking image checksum"
msgstr ""

#: pyanaconda/packaging/dnfpayload.py:176
#, python-format
msgid ""
"Downloading %(total_files)s RPMs, %(downloaded)s / %(total_size)s "
"(%(percent)d%%) done."
msgstr ""

#: pyanaconda/packaging/dnfpayload.py:764
msgid "Starting package installation process"
msgstr ""

#: pyanaconda/packaging/dnfpayload.py:782
msgid "Downloading packages"
msgstr ""

#: pyanaconda/packaging/dnfpayload.py:795
msgid "Preparing transaction from installation source"
msgstr ""

#: pyanaconda/packaging/dnfpayload.py:808
#, python-format
msgid "Installing %s"
msgstr ""

#: pyanaconda/packaging/rpmostreepayload.py:169
#, python-format
msgid "Starting pull of %(branchName)s from %(source)s"
msgstr ""

#: pyanaconda/packaging/rpmostreepayload.py:185
#, python-format
msgid "Preparing deployment of %s"
msgstr ""

#: pyanaconda/packaging/rpmostreepayload.py:205
#, python-format
msgid "Deployment starting: %s"
msgstr ""

#: pyanaconda/packaging/rpmostreepayload.py:208
#, python-format
msgid "Deployment complete: %s"
msgstr ""

#: pyanaconda/ui/helpers.py:92 pyanaconda/ui/tui/spokes/storage.py:408
msgid "Checking storage configuration..."
msgstr ""

#: pyanaconda/ui/categories/__init__.py:48
msgid "DEFAULT TITLE"
msgstr ""

#: pyanaconda/ui/categories/customization.py:32
msgid "CUSTOMIZATION"
msgstr ""

#: pyanaconda/ui/categories/localization.py:31
msgid "LOCALIZATION"
msgstr "YERELLEŞTİRME"

#: pyanaconda/ui/categories/software.py:31
msgid "SOFTWARE"
msgstr "YAZILIM"

#: pyanaconda/ui/categories/system.py:32
#: pyanaconda/ui/gui/spokes/lib/accordion.py:469
msgid "SYSTEM"
msgstr "SİSTEM"

#: pyanaconda/ui/categories/user_settings.py:32
msgid "USER SETTINGS"
msgstr ""

#: pyanaconda/ui/lib/space.py:37
#, python-format
msgid ""
"Not enough space in file systems for the current software selection. An "
"additional %s is needed."
msgstr ""

#: pyanaconda/ui/lib/entropy.py:58
msgid "The system needs more random data entropy"
msgstr ""

#: pyanaconda/ui/lib/entropy.py:67
msgid "Entropy can be increased by typing randomly on keyboard"
msgstr ""

#: pyanaconda/ui/lib/entropy.py:68
#, python-format
msgid ""
"After %d minutes, the installation will continue regardless of the amount of "
"available entropy"
msgstr ""

#: pyanaconda/ui/lib/entropy.py:92
#, python-format
msgid ""
"Available entropy: %(av_entr)s, Required entropy: %(req_entr)s [%(pct)d %%] "
"(%(rem)d %(min)s remaining)"
msgstr ""

#: pyanaconda/ui/lib/entropy.py:95
#: pyanaconda/ui/gui/spokes/lib/entropy_dialog.py:92
msgid "minute"
msgid_plural "minutes"
msgstr[0] ""

#: pyanaconda/ui/lib/entropy.py:102
#, python-format
msgid ""
"Available entropy: %(av_entr)s, Required entropy: %(req_entr)s [%(pct)d %%]"
msgstr ""

#: pyanaconda/ui/lib/entropy.py:107
msgid "Enough entropy gathered, please stop typing."
msgstr ""

#: pyanaconda/ui/lib/entropy.py:110
#, python-format
msgid "Giving up, time (%d minutes) ran out."
msgstr ""

#: pyanaconda/ui/lib/disks.py:112
#, python-format
msgid ""
"You selected disk %(selected)s, which contains devices that also use "
"unselected disk %(unselected)s. You must select or de-select these disks as "
"a set."
msgid_plural ""
"You selected disk %(selected)s, which contains devices that also use "
"unselected disks %(unselected)s. You must select or de-select these disks as "
"a set."
msgstr[0] ""

#: pyanaconda/ui/tui/hubs/summary.py:39
msgid "Installation"
msgstr ""

#: pyanaconda/ui/tui/hubs/summary.py:55
msgid "Starting automated install"
msgstr ""

#: pyanaconda/ui/tui/hubs/summary.py:88
msgid "The following mandatory spokes are not completed:"
msgstr ""

#: pyanaconda/ui/tui/hubs/summary.py:101
#, python-format
msgid ""
"  Please make your choice from above ['%(quit)s' to quit | '%(begin)s' to "
"begin installation |\n"
"  '%(refresh)s' to refresh]: "
msgstr ""

#. TRANSLATORS: 'q' to quit
#: pyanaconda/ui/tui/hubs/summary.py:103 pyanaconda/ui/tui/hubs/__init__.py:137
#: pyanaconda/ui/tui/simpleline/base.py:462
#: pyanaconda/ui/tui/simpleline/base.py:624
#: pyanaconda/ui/tui/spokes/askvnc.py:111
msgctxt "TUI|Spoke Navigation"
msgid "q"
msgstr ""

#. TRANSLATORS: 'b' to begin installation
#: pyanaconda/ui/tui/hubs/summary.py:105 pyanaconda/ui/tui/hubs/summary.py:122
msgctxt "TUI|Spoke Navigation"
msgid "b"
msgstr ""

#. TRANSLATORS: 'r' to refresh
#. TRANSLATORS:'r' to refresh
#. TRANSLATORS: 'r' to refresh
#. TRANSLATORS:'r' to refresh
#: pyanaconda/ui/tui/hubs/summary.py:107 pyanaconda/ui/tui/hubs/__init__.py:141
#: pyanaconda/ui/tui/simpleline/base.py:450
#: pyanaconda/ui/tui/simpleline/base.py:628
msgctxt "TUI|Spoke Navigation"
msgid "r"
msgstr ""

#: pyanaconda/ui/tui/hubs/summary.py:125 pyanaconda/ui/tui/hubs/__init__.py:116
msgid "Please complete all spokes before continuing"
msgstr ""

#. TRANSLATORS: 'c' to continue
#. TRANSLATORS:'c' to continue
#. TRANSLATORS: 'c' to continue
#. TRANSLATORS:'c' to continue
#. TRANSLATORS: 'c' to continue
#: pyanaconda/ui/tui/hubs/summary.py:136 pyanaconda/ui/tui/hubs/__init__.py:113
#: pyanaconda/ui/tui/hubs/__init__.py:139
#: pyanaconda/ui/tui/simpleline/base.py:456
#: pyanaconda/ui/tui/simpleline/base.py:626
#: pyanaconda/ui/tui/spokes/software.py:193
msgctxt "TUI|Spoke Navigation"
msgid "c"
msgstr ""

#: pyanaconda/ui/tui/hubs/__init__.py:43
msgid "Default HUB title"
msgstr ""

#: pyanaconda/ui/tui/hubs/__init__.py:133
#, python-format
msgid ""
"  Please make your choice from [ '1' to enter the %(spoke_title)s spoke | "
"'%(quit)s' to quit |\n"
"  '%(continue)s' to continue | '%(refresh)s' to refresh]: "
msgstr ""

#: pyanaconda/ui/tui/tuiobject.py:30 pyanaconda/ui/gui/main.glade:72
msgid "Error"
msgstr "Hata"

#: pyanaconda/ui/tui/tuiobject.py:51
msgid "Press Enter to exit."
msgstr ""

#: pyanaconda/ui/tui/tuiobject.py:61 pyanaconda/ui/tui/spokes/password.py:74
#: pyanaconda/ui/gui/spokes/password.glade:86
#: pyanaconda/ui/gui/spokes/user.glade:171
msgid "Password"
msgstr ""

#: pyanaconda/ui/tui/tuiobject.py:82
msgid "Passphrase: "
msgstr ""

#: pyanaconda/ui/tui/tuiobject.py:108
msgid "Question"
msgstr ""

#: pyanaconda/ui/tui/tuiobject.py:131
#, python-format
msgid "Please respond '%(yes)s' or '%(no)s': "
msgstr ""

#. TRANSLATORS: 'yes' as positive reply
#: pyanaconda/ui/tui/tuiobject.py:133 pyanaconda/ui/tui/tuiobject.py:140
msgctxt "TUI|Spoke Navigation"
msgid "yes"
msgstr ""

#. TRANSLATORS: 'no' as negative reply
#: pyanaconda/ui/tui/tuiobject.py:135 pyanaconda/ui/tui/tuiobject.py:146
msgctxt "TUI|Spoke Navigation"
msgid "no"
msgstr ""

#: pyanaconda/ui/tui/simpleline/base.py:87
msgid "Do you really want to quit?"
msgstr ""

#: pyanaconda/ui/tui/simpleline/base.py:622
#, python-format
msgid ""
"  Please make your choice from above ['%(quit)s' to quit | '%(continue)s' to "
"continue |\n"
"  '%(refresh)s' to refresh]: "
msgstr ""

#: pyanaconda/ui/tui/spokes/askvnc.py:46
msgid "VNC"
msgstr ""

#: pyanaconda/ui/tui/spokes/askvnc.py:62
msgid ""
"X was unable to start on your machine.  Would you like to start VNC to "
"connect to this computer from another computer and perform a graphical "
"installation or continue with a text mode installation?"
msgstr ""

#: pyanaconda/ui/tui/spokes/askvnc.py:131
msgid "VNC Password"
msgstr ""

#: pyanaconda/ui/tui/spokes/askvnc.py:139
msgid ""
"Please provide VNC password (must be six to eight characters long).\n"
"You will have to type it twice. Leave blank for no password"
msgstr ""

#: pyanaconda/ui/tui/spokes/askvnc.py:158
#: pyanaconda/ui/gui/spokes/network.py:1385
msgid "Password: "
msgstr ""

#: pyanaconda/ui/tui/spokes/askvnc.py:159
msgid "Password (confirm): "
msgstr ""

#: pyanaconda/ui/tui/spokes/askvnc.py:162
msgid "Passwords do not match!"
msgstr ""

#: pyanaconda/ui/tui/spokes/askvnc.py:165
msgid "The password must be at least six characters long."
msgstr "Parola en az altı karakter uzunluğunda olmalı."

#: pyanaconda/ui/tui/spokes/askvnc.py:169
msgid "The password cannot be more than eight characters long."
msgstr ""

#: pyanaconda/ui/tui/spokes/langsupport.py:41
msgid "Language settings"
msgstr ""

#: pyanaconda/ui/tui/spokes/langsupport.py:68
msgid "Language is not set."
msgstr ""

#: pyanaconda/ui/tui/spokes/langsupport.py:78
msgid "Available locales"
msgstr ""

#: pyanaconda/ui/tui/spokes/langsupport.py:81
msgid "Available languages"
msgstr ""

#. TRANSLATORS: 'b' to go back
#: pyanaconda/ui/tui/spokes/langsupport.py:117
#: pyanaconda/ui/tui/spokes/langsupport.py:127
msgctxt "TUI|Spoke Navigation|Language Support"
msgid "b"
msgstr ""

#: pyanaconda/ui/tui/spokes/langsupport.py:125
#, python-format
msgid ""
"Please select language support to install.\n"
"['%(back)s' to return to language list, '%(continue)s' to continue, "
"'%(quit)s' to quit]: "
msgstr ""

#. TRANSLATORS:'c' to continue
#: pyanaconda/ui/tui/spokes/langsupport.py:129
msgctxt "TUI|Spoke Navigation|Language Support"
msgid "c"
msgstr ""

#. TRANSLATORS:'q' to quit
#: pyanaconda/ui/tui/spokes/langsupport.py:131
msgctxt "TUI|Spoke Navigation|Language Support"
msgid "q"
msgstr ""

#: pyanaconda/ui/tui/spokes/network.py:51
msgid "Network configuration"
msgstr ""

#: pyanaconda/ui/tui/spokes/network.py:112
#, python-format
msgid "Wired (%(interface_name)s) disconnected\n"
msgstr ""

#: pyanaconda/ui/tui/spokes/network.py:117
#, python-format
msgid "Wired (%(interface_name)s) connected\n"
msgstr ""

#: pyanaconda/ui/tui/spokes/network.py:129
#, python-format
msgid " IPv4 Address: %(addr)s Netmask: %(netmask)s Gateway: %(gateway)s\n"
msgstr ""

#: pyanaconda/ui/tui/spokes/network.py:131
#, python-format
msgid " DNS: %s\n"
msgstr ""

#: pyanaconda/ui/tui/spokes/network.py:138
#, python-format
msgid " IPv6 Address: %(addr)s/%(prefix)d\n"
msgstr ""

#: pyanaconda/ui/tui/spokes/network.py:159
#, python-format
msgid "Host Name: %s\n"
msgstr ""

#: pyanaconda/ui/tui/spokes/network.py:171
msgid "Set host name"
msgstr ""

#: pyanaconda/ui/tui/spokes/network.py:173
#, python-format
msgid "Configure device %s"
msgstr ""

#: pyanaconda/ui/tui/spokes/network.py:192
#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:241
#: pyanaconda/ui/gui/spokes/network.glade:2201
msgid "Host Name"
msgstr ""

#: pyanaconda/ui/tui/spokes/network.py:212
msgid "Configuration of device not found"
msgstr ""

#: pyanaconda/ui/tui/spokes/network.py:225
msgid ""
"Configuration not saved: gateway or netmask missing in static configuration"
msgstr ""

#: pyanaconda/ui/tui/spokes/network.py:241
msgid "Can't apply configuration, device activation failed."
msgstr ""

#: pyanaconda/ui/tui/spokes/network.py:268
#: pyanaconda/ui/gui/spokes/network.py:1546
#: pyanaconda/ui/gui/spokes/network.py:1627
#, python-format
msgid "Host name is not valid: %s"
msgstr ""

#: pyanaconda/ui/tui/spokes/network.py:306
msgid "Device configuration"
msgstr ""

#: pyanaconda/ui/tui/spokes/network.py:310
#, python-format
msgid "IPv4 address or %s for DHCP"
msgstr ""

#: pyanaconda/ui/tui/spokes/network.py:312
msgid "IPv4 netmask"
msgstr ""

#: pyanaconda/ui/tui/spokes/network.py:313
msgid "IPv4 gateway"
msgstr ""

#: pyanaconda/ui/tui/spokes/network.py:314
#, python-format
msgid ""
"IPv6 address or %(auto)s for automatic, %(dhcp)s for DHCP, %(ignore)s to "
"turn off"
msgstr ""

#: pyanaconda/ui/tui/spokes/network.py:317
msgid "IPv6 default gateway"
msgstr ""

#: pyanaconda/ui/tui/spokes/network.py:318
msgid "Nameservers (comma separated)"
msgstr ""

#: pyanaconda/ui/tui/spokes/network.py:320
msgid "Connect automatically after reboot"
msgstr ""

#: pyanaconda/ui/tui/spokes/network.py:321
msgid "Apply configuration in installer"
msgstr ""

#: pyanaconda/ui/tui/spokes/network.py:332
msgid "Bad format of the IP address"
msgstr ""

#: pyanaconda/ui/tui/spokes/network.py:337
#, python-format
msgid "Configuring device %s."
msgstr ""

#: pyanaconda/ui/tui/spokes/password.py:36
msgid "Root password"
msgstr ""

#: pyanaconda/ui/tui/spokes/password.py:59
msgid "Password is set."
msgstr ""

#: pyanaconda/ui/tui/spokes/password.py:61
msgid "Root account is disabled."
msgstr ""

#: pyanaconda/ui/tui/spokes/password.py:63
msgid "Password is not set."
msgstr ""

#: pyanaconda/ui/tui/spokes/password.py:68
msgid "Please select new root password. You will have to type it twice."
msgstr ""

#: pyanaconda/ui/tui/spokes/storage.py:60
msgid "Use All Space"
msgstr ""

#: pyanaconda/ui/tui/spokes/storage.py:61
msgid "Replace Existing Linux system(s)"
msgstr ""

#: pyanaconda/ui/tui/spokes/storage.py:62
msgid "Use Free Space"
msgstr ""

#: pyanaconda/ui/tui/spokes/storage.py:74
msgid "Installation Destination"
msgstr ""

#: pyanaconda/ui/tui/spokes/storage.py:130
#: pyanaconda/ui/gui/spokes/storage.py:452
msgid "No disks selected"
msgstr "Hiçbir disk seçilmedi."

#: pyanaconda/ui/tui/spokes/storage.py:133
#: pyanaconda/ui/gui/spokes/storage.py:455
msgid "Kickstart insufficient"
msgstr ""

#: pyanaconda/ui/tui/spokes/storage.py:140
#: pyanaconda/ui/gui/spokes/storage.py:464
msgid "Error checking storage configuration"
msgstr ""

#: pyanaconda/ui/tui/spokes/storage.py:142
#: pyanaconda/ui/gui/spokes/storage.py:466
msgid "Warning checking storage configuration"
msgstr ""

#: pyanaconda/ui/tui/spokes/storage.py:145
#: pyanaconda/ui/gui/spokes/storage.py:468
msgid "Automatic partitioning selected"
msgstr ""

#: pyanaconda/ui/tui/spokes/storage.py:147
#: pyanaconda/ui/gui/spokes/storage.py:470
msgid "Custom partitioning selected"
msgstr ""

#: pyanaconda/ui/tui/spokes/storage.py:183
#: pyanaconda/ui/gui/spokes/storage.py:725
msgid ""
"No disks detected.  Please shut down the computer, connect at least one "
"disk, and restart to complete installation."
msgstr ""

#: pyanaconda/ui/tui/spokes/storage.py:185
#: pyanaconda/ui/gui/spokes/storage.py:727
msgid "No disks selected; please select at least one disk to install to."
msgstr ""

#: pyanaconda/ui/tui/spokes/storage.py:221
msgid "Select all"
msgstr ""

#: pyanaconda/ui/tui/spokes/storage.py:326
msgid ""
"The following unformatted DASDs have been detected on your system. You can "
"choose to format them now with dasdfmt or cancel to leave them unformatted. "
"Unformatted DASDs cannot be used during installation.\n"
"\n"
msgstr ""

#: pyanaconda/ui/tui/spokes/storage.py:328
msgid ""
"Warning: All storage changes made using the installer will be lost when you "
"choose to format.\n"
"\n"
"Proceed to run dasdfmt?\n"
msgstr ""

#: pyanaconda/ui/tui/spokes/storage.py:343
#: pyanaconda/ui/gui/spokes/lib/dasdfmt.py:95
#, python-format
msgid "Formatting /dev/%s. This may take a moment."
msgstr ""

#: pyanaconda/ui/tui/spokes/storage.py:387
msgid "Generating updated storage configuration"
msgstr ""

#: pyanaconda/ui/tui/spokes/storage.py:392
#: pyanaconda/ui/tui/spokes/storage.py:404
#, python-format
msgid "storage configuration failed: %s"
msgstr ""

#: pyanaconda/ui/tui/spokes/storage.py:455
msgid "Autopartitioning Options"
msgstr ""

#: pyanaconda/ui/tui/spokes/storage.py:481
msgid ""
"Installation requires partitioning of your hard drive. Select what space to "
"use for the install target."
msgstr ""

#: pyanaconda/ui/tui/spokes/storage.py:520
msgid "Partition Scheme Options"
msgstr ""

#: pyanaconda/ui/tui/spokes/storage.py:544
msgid "Select a partition scheme configuration."
msgstr ""

#: pyanaconda/ui/tui/spokes/time_spoke.py:35
msgid "Timezone settings"
msgstr ""

#: pyanaconda/ui/tui/spokes/time_spoke.py:65
#: pyanaconda/ui/gui/spokes/datetime_spoke.py:550
#: pyanaconda/ui/gui/spokes/datetime_spoke.py:556
#, python-format
msgid "%s timezone"
msgstr "zaman dilimi %s"

#: pyanaconda/ui/tui/spokes/time_spoke.py:67
msgid "Timezone is not set."
msgstr ""

#: pyanaconda/ui/tui/spokes/time_spoke.py:75
#, python-format
msgid "Available timezones in region %s"
msgstr ""

#: pyanaconda/ui/tui/spokes/time_spoke.py:78
msgid "Available regions"
msgstr ""

#. TRANSLATORS: 'b' to go back
#: pyanaconda/ui/tui/spokes/time_spoke.py:117
#: pyanaconda/ui/tui/spokes/time_spoke.py:147
msgctxt "TUI|Spoke Navigation|Time Settings"
msgid "b"
msgstr ""

#: pyanaconda/ui/tui/spokes/time_spoke.py:145
#, python-format
msgid ""
"Please select the timezone.\n"
"Use numbers or type names directly ['%(back)s' to region list, '%(quit)s' to "
"quit]: "
msgstr ""

#. TRANSLATORS:'q' to quit
#: pyanaconda/ui/tui/spokes/time_spoke.py:149
msgctxt "TUI|Spoke Navigation|Time Settings"
msgid "q"
msgstr ""

#: pyanaconda/ui/tui/spokes/shell_spoke.py:37
msgid "Shell"
msgstr ""

#: pyanaconda/ui/tui/spokes/shell_spoke.py:52
msgid "Start shell"
msgstr ""

#: pyanaconda/ui/tui/spokes/shell_spoke.py:61
msgid "Exit the shell to continue"
msgstr ""

#: pyanaconda/ui/tui/spokes/source.py:59
msgid "Installation source"
msgstr ""

#: pyanaconda/ui/tui/spokes/source.py:62 pyanaconda/ui/tui/spokes/source.py:111
#: pyanaconda/ui/gui/spokes/source.py:600
#: pyanaconda/ui/gui/spokes/source.glade:758
msgid "Closest mirror"
msgstr "En yakın ayna"

#: pyanaconda/ui/tui/spokes/source.py:103
#: pyanaconda/ui/gui/spokes/source.py:592
#, python-format
msgid "NFS server %s"
msgstr ""

#: pyanaconda/ui/tui/spokes/source.py:105
#: pyanaconda/ui/gui/spokes/source.py:594
msgid "Local media"
msgstr ""

#: pyanaconda/ui/tui/spokes/source.py:108
#: pyanaconda/ui/tui/spokes/source.py:122
#: pyanaconda/ui/gui/spokes/source.py:588
msgid "Error setting up software source"
msgstr ""

#: pyanaconda/ui/tui/spokes/source.py:113
#: pyanaconda/ui/tui/spokes/software.py:125
#: pyanaconda/ui/gui/spokes/datetime_spoke.py:558
#: pyanaconda/ui/gui/spokes/software.py:310
#: pyanaconda/ui/gui/spokes/software.py:315
#: pyanaconda/ui/gui/spokes/source.py:602
msgid "Nothing selected"
msgstr ""

#: pyanaconda/ui/tui/spokes/source.py:124
#: pyanaconda/ui/tui/spokes/software.py:113
msgid "Processing..."
msgstr ""

#: pyanaconda/ui/tui/spokes/source.py:140
msgid "CD/DVD"
msgstr ""

#: pyanaconda/ui/tui/spokes/source.py:140
msgid "local ISO file"
msgstr ""

#: pyanaconda/ui/tui/spokes/source.py:140
msgid "Network"
msgstr ""

#: pyanaconda/ui/tui/spokes/source.py:144
#: pyanaconda/ui/gui/spokes/source.py:913
msgid ""
"The installation source is in use by the installer and cannot be changed."
msgstr ""

#: pyanaconda/ui/tui/spokes/source.py:151
msgid "Choose an installation source type."
msgstr ""

#: pyanaconda/ui/tui/spokes/source.py:248
#: pyanaconda/ui/tui/spokes/source.py:286
msgid "Specify Repo Options"
msgstr ""

#: pyanaconda/ui/tui/spokes/source.py:252
msgid "Repo URL"
msgstr ""

#: pyanaconda/ui/tui/spokes/source.py:290
msgid "<server>:/<path>"
msgstr ""

#: pyanaconda/ui/tui/spokes/source.py:291
msgid "NFS mount options"
msgstr ""

#: pyanaconda/ui/tui/spokes/source.py:333
msgid "Select device containing the ISO file"
msgstr ""

#: pyanaconda/ui/tui/spokes/source.py:392
msgid "No mountable devices found"
msgstr ""

#: pyanaconda/ui/tui/spokes/source.py:420
msgid "Select an ISO to use as install source"
msgstr ""

#: pyanaconda/ui/tui/spokes/source.py:449
msgid "No *.iso files found in device root folder"
msgstr ""

#: pyanaconda/ui/tui/spokes/software.py:44
msgid "Software selection"
msgstr ""

#: pyanaconda/ui/tui/spokes/software.py:111
#: pyanaconda/ui/gui/spokes/software.py:291
msgid "Error checking software selection"
msgstr ""

#: pyanaconda/ui/tui/spokes/software.py:115
#: pyanaconda/ui/gui/spokes/software.py:294
msgid "Installation source not set up"
msgstr ""

#: pyanaconda/ui/tui/spokes/software.py:117
#: pyanaconda/ui/gui/spokes/software.py:297
msgid "Source changed - please verify"
msgstr ""

#: pyanaconda/ui/tui/spokes/software.py:124
#: pyanaconda/ui/gui/spokes/software.py:304
msgid "Custom software selected"
msgstr ""

#: pyanaconda/ui/tui/spokes/software.py:151
msgid "Installation source needs to be set up first."
msgstr ""

#: pyanaconda/ui/tui/spokes/software.py:161
msgid "Base environment"
msgstr ""

#: pyanaconda/ui/tui/spokes/user.py:41
msgid "User creation"
msgstr ""

#: pyanaconda/ui/tui/spokes/user.py:74
msgid ""
"You have provided an invalid user name.\n"
"Tip: Keep your user name shorter than 32 characters and do not use spaces.\n"
msgstr ""

#: pyanaconda/ui/tui/spokes/user.py:129 pyanaconda/ui/gui/spokes/user.py:350
msgid "No user will be created"
msgstr ""

#: pyanaconda/ui/tui/spokes/user.py:131
msgid "You must set a password"
msgstr ""

#: pyanaconda/ui/tui/spokes/user.py:133 pyanaconda/ui/gui/spokes/user.py:352
#, python-format
msgid "Administrator %s will be created"
msgstr ""

#: pyanaconda/ui/tui/spokes/user.py:135 pyanaconda/ui/gui/spokes/user.py:354
#, python-format
msgid "User %s will be created"
msgstr ""

#: pyanaconda/ui/tui/spokes/user.py:141
#, python-format
msgid "Invalid user name: %s.\n"
msgstr ""

#: pyanaconda/ui/tui/spokes/warnings_spoke.py:40
msgid "Warnings"
msgstr ""

#: pyanaconda/ui/tui/spokes/warnings_spoke.py:48
#: pyanaconda/ui/gui/spokes/welcome.glade:510
msgid ""
"This hardware (or a combination thereof) is not supported by Red Hat.  For "
"more information on supported hardware, please refer to http://www.redhat."
"com/hardware."
msgstr ""

#: pyanaconda/ui/tui/spokes/progress.py:41
msgid "Progress"
msgstr ""

#: pyanaconda/ui/tui/spokes/progress.py:143
msgid "\tInstallation complete.  Press Enter to quit"
msgstr ""

#: pyanaconda/ui/tui/spokes/__init__.py:52
msgid "Default spoke title"
msgstr ""

#: pyanaconda/ui/tui/spokes/__init__.py:64
msgid "testing status..."
msgstr ""

#: pyanaconda/ui/tui/spokes/__init__.py:117
msgid "New value"
msgstr ""

#: pyanaconda/ui/tui/spokes/__init__.py:144
#, python-format
msgid "%s: "
msgstr ""

#: pyanaconda/ui/tui/spokes/__init__.py:145
#, python-format
msgid "%s (confirm): "
msgstr ""

#: pyanaconda/ui/tui/spokes/__init__.py:148
msgid ""
"You must enter your root password and confirm it by typing it a second time "
"to continue."
msgstr ""

#: pyanaconda/ui/tui/spokes/__init__.py:170
msgid ""
"\n"
"Would you like to use it anyway?"
msgstr ""

#: pyanaconda/ui/tui/spokes/__init__.py:187
msgid ""
"You have provided a password containing non-ASCII characters.\n"
"You may not be able to switch between keyboard layouts to login.\n"
msgstr ""

#: pyanaconda/ui/tui/spokes/__init__.py:193
#, python-format
msgid "Enter a new value for '%s' and press <Enter>\n"
msgstr ""

#: pyanaconda/ui/tui/spokes/__init__.py:204
msgid "You have provided an invalid value\n"
msgstr ""

#: pyanaconda/ui/tui/spokes/__init__.py:317
msgid "Password set."
msgstr ""

#: pyanaconda/ui/gui/__init__.py:447 widgets/src/BaseWindow.c:153
msgid "Help!"
msgstr "Yardım!"

#: pyanaconda/ui/gui/__init__.py:893
#: pyanaconda/ui/gui/spokes/lib/detailederror.py:59
msgctxt "GUI|Detailed Error Dialog"
msgid "_Quit"
msgstr "Çık"

#: pyanaconda/ui/gui/__init__.py:909
msgctxt "GUI|Yes No Dialog"
msgid "_No"
msgstr ""

#: pyanaconda/ui/gui/__init__.py:910
msgctxt "GUI|Yes No Dialog"
msgid "_Yes"
msgstr ""

#: pyanaconda/ui/gui/hubs/__init__.py:241
msgid ""
"Please complete items marked with this icon before continuing to the next "
"step."
msgstr ""

#: pyanaconda/ui/gui/hubs/progress.py:119
msgid "Complete!"
msgstr ""

#: pyanaconda/ui/gui/hubs/progress.py:219
#, python-format
msgid ""
"%s is now successfully installed on your system and ready for you to use!  "
"When you are ready, reboot your system to start using it!"
msgstr ""

#: pyanaconda/ui/gui/hubs/progress.py:222
msgctxt "GUI|Progress"
msgid "_Quit"
msgstr "Çık"

#: pyanaconda/ui/gui/hubs/progress.py:230
#, python-format
msgid ""
"%s is now successfully installed, but some configuration still needs to be "
"done.\n"
"Finish it and then click the Finish configuration button please."
msgstr ""

#: pyanaconda/ui/gui/hubs/progress.py:235
#, python-format
msgid ""
"%s is now successfully installed and ready for you to use!\n"
"Go ahead and reboot to start using it!"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.py:116
#, python-format
msgid "Create a new %(container_type)s ..."
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.py:117
#, python-format
msgid "Create or select %(container_type)s"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.py:119
msgid "Device reconfiguration failed. <a href=\"\">Click for details.</a>"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.py:121
msgid ""
"Storage configuration reset due to unrecoverable error. <a href=\"\">Click "
"for details.</a>"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.py:156
#: pyanaconda/ui/gui/spokes/custom.glade:99
msgid "MANUAL PARTITIONING"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.py:382
#, python-format
msgctxt "GUI|Custom Partitioning"
msgid "%d _storage device selected"
msgid_plural "%d _storage devices selected"
msgstr[0] ""

#: pyanaconda/ui/gui/spokes/custom.py:474
#, python-format
msgid "%(items_selected)s of %(items_total)s mount points in %(page_name)s"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.py:500
#, python-format
msgid ""
"Please select a single mount point to edit properties.\n"
"\n"
"You have currently selected:\n"
"%s"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.py:504
#, python-format
msgid ""
"When you create mount points for your %(name)s %(version)s installation, "
"you'll be able to view their details here."
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.py:689
#, python-format
msgid ""
"/boot/efi must be on a device of type %(oneFsType)s or %(anotherFsType)s"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.py:693
#, python-format
msgid "%(fs)s must be on a device of type %(type)s"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.py:696
#: pyanaconda/ui/gui/spokes/custom.py:698
#, python-format
msgid "%s cannot be encrypted"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.py:700
msgid "You must create a new file system on the root device."
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.py:705
#, python-format
msgid "Device does not support RAID level selection %s."
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.py:818
msgid "Device resize request failed. <a href=\"\">Click for details.</a>"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.py:891
msgid "Device reformat request failed. <a href=\"\">Click for details.</a>"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.py:1262
#: pyanaconda/ui/gui/spokes/custom.py:2208
#, python-format
msgid "Invalid device name: %s"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.py:1268
#, python-format
msgid "Specified name %s already in use."
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.py:1475
msgid "No disks assigned"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.py:1481
#, python-format
msgctxt "GUI|Custom Partitioning|Devices"
msgid " and %d other"
msgid_plural " and %d others"
msgstr[0] ""

#: pyanaconda/ui/gui/spokes/custom.py:1539
msgid "The container is encrypted."
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.py:1575
msgid ""
"The space available to this mount point can be changed by modifying the "
"volume below."
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.py:1577
msgid "This file system may not be resized."
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.py:1631
msgid ""
"Error checking storage configuration.  <a href=\"\">Click for details</a> or "
"press Done again to continue."
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.py:1633
msgid ""
"Warning checking storage configuration.  <a href=\"\">Click for details</a> "
"or press Done again to continue."
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.py:1739
#, python-format
msgid "Added new %(type)s to existing container %(name)s."
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.py:1750
msgid "Failed to add new device. <a href=\"\">Click for details.</a>"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.py:1755
msgid "Invalid partition size set. Use a valid integer."
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.py:1862
msgid "Device removal request failed. <a href=\"\">Click for details.</a>"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.py:1945
#, python-format
msgctxt "GUI|Custom Partitioning|Confirm Delete Dialog"
msgid "Delete _all file systems which are only used by %s."
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.py:1949
msgctxt "GUI|Custom Partitioning|Confirm Delete Dialog"
msgid "Do _not show this dialog for other selected file systems."
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.py:2145
#, python-format
msgid "Volume Group name %s is already in use. Not saving changes."
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.py:2175
#, python-format
msgid "(%s free)"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.py:2346
#, python-format
msgid ""
"This Software RAID array is missing %(missingMembers)d of %(totalMembers)d "
"member partitions. You can remove it or select a different device."
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.py:2352
#, python-format
msgid ""
"This LVM Volume Group is missing %(missingPVs)d of %(totalPVs)d physical "
"volumes. You can remove it or select a different device."
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.py:2356
#, python-format
msgid ""
"This %(type)s device is missing member devices. You can remove it or select "
"a different device."
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.py:2438
msgid "No disks selected."
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.py:2443
msgid "Not enough free space on selected disks."
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.py:2448
#: pyanaconda/ui/gui/spokes/custom.py:2473
msgid "Automatic partitioning failed. <a href=\"\">Click for details.</a>"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.py:2703
msgid ""
"Continuing with this action will reset all your partitioning selections to "
"their current on-disk state."
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.py:2711
msgctxt "GUI|Custom Partitioning|Reset Dialog"
msgid "_Reset selections"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.py:2712
msgctxt "GUI|Custom Partitioning|Reset Dialog"
msgid "_Preserve current selections"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.py:2787
msgid ""
"Failed to unlock encrypted block device. <a href=\"\">Click for details.</a>"
msgstr ""

#: pyanaconda/ui/gui/spokes/datetime_spoke.py:418
msgctxt "GUI|Spoke"
msgid "_TIME & DATE"
msgstr ""

#: pyanaconda/ui/gui/spokes/datetime_spoke.py:541
msgid "Restoring hardware time..."
msgstr ""

#: pyanaconda/ui/gui/spokes/datetime_spoke.py:552
msgid "Invalid timezone"
msgstr ""

#: pyanaconda/ui/gui/spokes/datetime_spoke.py:727
#: pyanaconda/ui/gui/spokes/datetime_spoke.py:741
#: pyanaconda/ui/gui/spokes/datetime_spoke.py:927
#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:751
msgid "PM"
msgstr ""

#: pyanaconda/ui/gui/spokes/datetime_spoke.py:729
#: pyanaconda/ui/gui/spokes/datetime_spoke.py:738
#: pyanaconda/ui/gui/spokes/datetime_spoke.py:926
#: pyanaconda/ui/gui/spokes/datetime_spoke.py:929
msgid "AM"
msgstr ""

#: pyanaconda/ui/gui/spokes/datetime_spoke.py:1082
msgid "You need to set up networking first if you want to use NTP"
msgstr ""

#: pyanaconda/ui/gui/spokes/datetime_spoke.py:1086
msgid "You have no working NTP server configured"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.py:490
msgctxt "GUI|Spoke"
msgid "_INSTALLATION DESTINATION"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.py:597
#, python-format
msgctxt "GUI|Installation Destination|Filter"
msgid "%d _storage device selected"
msgid_plural "%d _storage devices selected"
msgstr[0] ""

#: pyanaconda/ui/gui/spokes/keyboard.py:52
#, python-format
msgid "%s to switch layouts."
msgstr ""

#: pyanaconda/ui/gui/spokes/keyboard.py:276
msgctxt "GUI|Spoke"
msgid "_KEYBOARD"
msgstr "KLAVYE"

#: pyanaconda/ui/gui/spokes/keyboard.py:363
msgid "Testing layouts configuration not available."
msgstr ""

#: pyanaconda/ui/gui/spokes/keyboard.py:371
msgid "Getting list of layouts..."
msgstr ""

#: pyanaconda/ui/gui/spokes/keyboard.py:426
msgid "Layout switching not configured."
msgstr ""

#: pyanaconda/ui/gui/spokes/langsupport.py:63
msgctxt "GUI|Spoke"
msgid "_LANGUAGE SUPPORT"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.py:79
msgid "Status unknown (missing)"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.py:82
msgid "Status unknown"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.py:84
msgid "Unmanaged"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.py:87
#: pyanaconda/ui/gui/spokes/network.py:91
msgid "Unavailable"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.py:89
msgid "Firmware missing"
msgstr "Yazılım eksik"

#: pyanaconda/ui/gui/spokes/network.py:95
msgid "Cable unplugged"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.py:97
msgid "Disconnected"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.py:102
msgid "Connecting"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.py:104
msgid "Authentication required"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.py:106
msgid "Connected"
msgstr "Bağlandı"

#: pyanaconda/ui/gui/spokes/network.py:108
msgid "Disconnecting"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.py:110
msgid "Connection failed"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.py:274
msgid "Ethernet"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.py:275
msgid "Wireless"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.py:276
#: pyanaconda/ui/gui/spokes/network.glade:15
msgid "Bond"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.py:277
#: pyanaconda/ui/gui/spokes/network.glade:27
msgid "VLAN"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.py:278
#: pyanaconda/ui/gui/spokes/network.glade:23
msgid "Team"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.py:279
#: pyanaconda/ui/gui/spokes/network.glade:19
msgid "Bridge"
msgstr ""

#. TRANSLATORS: ethernet cable is unplugged
#: pyanaconda/ui/gui/spokes/network.py:809
msgid "unplugged"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.py:897
#: pyanaconda/ui/gui/spokes/network.glade:519
#: pyanaconda/ui/gui/spokes/network.glade:795
msgid "IPv4 Address"
msgstr "IPv4  Adresi"

#: pyanaconda/ui/gui/spokes/network.py:898
#: pyanaconda/ui/gui/spokes/network.glade:494
#: pyanaconda/ui/gui/spokes/network.glade:807
#: pyanaconda/ui/gui/spokes/network.glade:1374
msgid "IPv6 Address"
msgstr "IPv6 Adresi"

#: pyanaconda/ui/gui/spokes/network.py:900
#: pyanaconda/ui/gui/spokes/network.py:902
#: pyanaconda/ui/gui/spokes/network.glade:1362
msgid "IP Address"
msgstr "IP Adresi"

#: pyanaconda/ui/gui/spokes/network.py:985
#, python-format
msgid "%d Mb/s"
msgstr "%d Mb/s"

#: pyanaconda/ui/gui/spokes/network.py:1162
msgid "WEP"
msgstr "WEP"

#: pyanaconda/ui/gui/spokes/network.py:1165
msgid "WPA"
msgstr "WPA"

#: pyanaconda/ui/gui/spokes/network.py:1168
msgid "WPA2"
msgstr "WPA2"

#: pyanaconda/ui/gui/spokes/network.py:1172
msgid "Enterprise"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.py:1177 widgets/src/SpokeSelector.c:83
#: pyanaconda/ui/gui/spokes/filter.glade:145
#: pyanaconda/ui/gui/spokes/filter.glade:560
#: pyanaconda/ui/gui/spokes/filter.glade:894
#: pyanaconda/ui/gui/spokes/filter.glade:1214
#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:535
msgid "None"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.py:1347
#: pyanaconda/ui/gui/spokes/network.glade:2403
msgid "Authentication required by wireless network"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.py:1348
#, python-format
msgid ""
"Passwords or encryption keys are required to access\n"
"the wireless network '%(network_id)s'."
msgstr ""

#: pyanaconda/ui/gui/spokes/network.py:1362
msgctxt "GUI|Network|Secrets Dialog"
msgid "_Password:"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.py:1370
msgctxt "GUI|Network|Secrets Dialog"
msgid "_Key:"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.py:1380
msgid "User name: "
msgstr ""

#: pyanaconda/ui/gui/spokes/network.py:1391
msgid "Identity: "
msgstr ""

#: pyanaconda/ui/gui/spokes/network.py:1396
msgid "Private key password: "
msgstr ""

#: pyanaconda/ui/gui/spokes/network.py:1456
msgctxt "GUI|Spoke"
msgid "_NETWORK & HOST NAME"
msgstr ""

#: pyanaconda/ui/gui/spokes/password.py:56
msgctxt "GUI|Spoke"
msgid "_ROOT PASSWORD"
msgstr ""

#: pyanaconda/ui/gui/spokes/password.py:99
#: pyanaconda/ui/gui/spokes/password.py:100
msgid "The password is set."
msgstr ""

#: pyanaconda/ui/gui/spokes/password.py:128
msgid "Root password is set"
msgstr ""

#: pyanaconda/ui/gui/spokes/password.py:130
msgid "Root account is disabled"
msgstr ""

#: pyanaconda/ui/gui/spokes/password.py:132
msgid "Root password is not set"
msgstr ""

#: pyanaconda/ui/gui/spokes/software.py:60
msgctxt "GUI|Spoke"
msgid "_SOFTWARE SELECTION"
msgstr ""

#: pyanaconda/ui/gui/spokes/software.py:217
#: pyanaconda/ui/gui/spokes/source.py:582
msgid "Checking software dependencies..."
msgstr ""

#: pyanaconda/ui/gui/spokes/software.py:222
msgid "Error checking software dependencies"
msgstr ""

#: pyanaconda/ui/gui/spokes/software.py:307
msgid "Invalid environment specified in kickstart"
msgstr ""

#: pyanaconda/ui/gui/spokes/software.py:320
msgid "Selected environment is not valid"
msgstr ""

#: pyanaconda/ui/gui/spokes/software.py:475
msgid ""
"Error checking software dependencies.  <a href=\"\">Click for details.</a>"
msgstr ""

#: pyanaconda/ui/gui/spokes/software.py:582
msgid ""
"The software marked for installation has the following errors.  This is "
"likely caused by an error with your installation source.  You can quit the "
"installer, change your software source, or change your software selections."
msgstr ""

#: pyanaconda/ui/gui/spokes/software.py:587
msgctxt "GUI|Software Selection|Error Dialog"
msgid "_Quit"
msgstr "Çık"

#: pyanaconda/ui/gui/spokes/software.py:588
msgctxt "GUI|Software Selection|Error Dialog"
msgid "_Modify Software Source"
msgstr ""

#: pyanaconda/ui/gui/spokes/software.py:589
msgctxt "GUI|Software Selection|Error Dialog"
msgid "Modify _Selections"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.py:62
msgid "Setting up installation source..."
msgstr ""

#: pyanaconda/ui/gui/spokes/source.py:82
msgid " <a href=\"\">Click for details.</a>"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.py:93
msgid "Invalid proxy URL"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.py:98
#, python-format
msgid "Invalid proxy protocol: %s"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.py:104
msgid "Extra characters in proxy URL"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.py:108
msgid "Proxy authentication data duplicated"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.py:262
msgid "This media is good to install from."
msgstr ""

#: pyanaconda/ui/gui/spokes/source.py:264
msgid "This media is not good to install from."
msgstr ""

#: pyanaconda/ui/gui/spokes/source.py:398
msgctxt "GUI|Spoke"
msgid "_INSTALLATION SOURCE"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.py:520
msgid "Failed to set up installation source; check the repo url"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.py:586
msgid "Error setting up base repository"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.py:597
msgid "Error setting up ISO file"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.py:730
msgid "Failed to set up installation source; check the repo url."
msgstr ""

#: pyanaconda/ui/gui/spokes/source.py:732
msgid ""
"Failed to set up installation source; check the repo url and proxy settings."
msgstr ""

#: pyanaconda/ui/gui/spokes/source.py:764
#, python-format
msgid "Device: %s"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.py:765
#, python-format
msgid "Label: %s"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.py:925
msgid "You need to configure the network to use a network installation source."
msgstr ""

#: pyanaconda/ui/gui/spokes/source.py:996
#, python-format
msgid "Repository %s has empty url"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.py:998
msgid "URL is empty"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.py:1003
#, python-format
msgid "Repository %s has invalid url"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.py:1005
msgid "Invalid URL"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.py:1012
#, python-format
msgid "Repository %s does not match selected protocol"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.py:1014
msgid "Protocol in URL does not match selected protocol"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.py:1018
#, python-format
msgid "Repository %s has empty NFS server"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.py:1020
msgid "NFS server is empty"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.py:1028
#, python-format
msgid "Repository %s has invalid host name"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.py:1030
msgid "Invalid host name"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.py:1034
#, python-format
msgid "Repository %s required remote directory"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.py:1036
msgid "Remote directory is required"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.py:1057
msgid "Duplicate repository names."
msgstr ""

#: pyanaconda/ui/gui/spokes/source.py:1065
msgid "Empty repository name"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.py:1068
msgid "Invalid repository name"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.py:1074
msgid "Repository name conflicts with internal repository name."
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.py:123
msgid "Please wait... software metadata still loading."
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.py:126
#, python-format
msgid ""
"Your current <b>%(product)s</b> software selection requires <b>%(total)s</b> "
"of available space, including <b>%(software)s</b> for software and <b>"
"%(swap)s</b> for swap space."
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.py:135
#, python-format
msgid ""
"Your current <a href=\"\" title=\"%(tooltip)s\"><b>%(product)s</b> software "
"selection</a> requires <b>%(total)s</b> of available space, including <b>"
"%(software)s</b> for software and <b>%(swap)s</b> for swap space."
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.py:185
#, python-format
msgid "%s The disks you've selected have the following amounts of free space:"
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.py:195
#, python-format
msgid ""
"<b>You don't have enough space available to install %s</b>.  You can shrink "
"or remove existing partitions via our guided reclaim space tool, or you can "
"adjust your partitions on your own in the custom partitioning interface."
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.py:219
#, python-format
msgid ""
"  You don't have enough space available to install <b>%(product)s</b>, even "
"if you used all of the free space available on the selected disks."
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.py:231
#, python-format
msgid ""
"<b>You don't have enough space available to install %(productName)s</b>, "
"even if you used all of the free space available on the selected disks.  You "
"could add more disks for additional space, modify your software selection to "
"install a smaller version of <b>%(productName)s</b>, or quit the installer."
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.py:256
msgctxt "GUI|Spoke"
msgid "INSTALLATION _DESTINATION"
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.py:373
msgid "Saving storage configuration..."
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.py:376
msgid "Passphrase for autopart encryption not specified."
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.py:385
#: pyanaconda/ui/gui/spokes/storage.py:397
msgid "Failed to save storage configuration..."
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.py:457
#: pyanaconda/ui/gui/spokes/storage.py:758
msgid "Formatting DASDs"
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.py:607
msgid ""
"Error checking storage configuration.  <a href=\"\">Click for details.</a>"
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.py:609
msgid ""
"Warning checking storage configuration.  <a href=\"\">Click for details.</a>"
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.py:650
#, python-format
msgid ""
"FCP device %(hba_id)s\n"
"WWPN %(wwpn)s\n"
"LUN %(lun)s"
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.py:660
#, python-format
msgid "%s free"
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.py:707
#, python-format
msgid "%(count)d disk selected; %(capacity)s capacity; %(free)s free"
msgid_plural "%(count)d disks selected; %(capacity)s capacity; %(free)s free"
msgstr[0] ""

#: pyanaconda/ui/gui/spokes/storage.py:791
msgid ""
"You have chosen to skip boot loader installation.  Your system may not be "
"bootable."
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.py:964
msgid "There was a problem with your disk selection. Click here for details."
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.py:1057
msgid "You'll be able to make space available during custom partitioning."
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.py:1077
msgid ""
"The following errors were encountered when checking your disk selection. You "
"can modify your selection or quit the installer."
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.py:1082
#: pyanaconda/ui/gui/spokes/storage.py:1103
msgctxt "GUI|Storage|Error Dialog"
msgid "_Quit"
msgstr "Çık"

#: pyanaconda/ui/gui/spokes/storage.py:1083
msgctxt "GUI|Storage|Error Dialog"
msgid "_Modify Disk Selection"
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.py:1098
msgid ""
"The following errors were encountered when checking your storage "
"configuration.  You can modify your storage layout or quit the installer."
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.py:1104
msgctxt "GUI|Storage|Error Dialog"
msgid "_Modify Storage Layout"
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.py:1118
msgid ""
"The following warnings were encountered when checking your storage "
"configuration.  These are not fatal, but you may wish to make changes to "
"your storage layout."
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.py:1123
msgctxt "GUI|Storage|Warning Dialog"
msgid "_OK"
msgstr ""

#: pyanaconda/ui/gui/spokes/user.py:67
#, python-format
msgid "Invalid group name: %s"
msgstr ""

#: pyanaconda/ui/gui/spokes/user.py:228
msgctxt "GUI|Spoke"
msgid "_USER CREATION"
msgstr ""

#: pyanaconda/ui/gui/spokes/user.py:313
msgid "Invalid user name"
msgstr ""

#: pyanaconda/ui/gui/spokes/user.py:315
msgid "Full name cannot contain colon characters"
msgstr ""

#: pyanaconda/ui/gui/spokes/user.py:322 pyanaconda/ui/gui/spokes/user.py:323
msgid "The password was set by kickstart."
msgstr ""

#: pyanaconda/ui/gui/spokes/welcome.py:262
#, python-format
msgid "WELCOME TO %(name)s %(version)s."
msgstr "%(name)s %(version)s'E HOŞGELDİNİZ"

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.py:283
#, python-format
msgid ""
"The following nodes were discovered using the iSCSI initiator <b>"
"%(initiatorName)s</b> using the target IP address <b>%(targetAddress)s</b>.  "
"Please select which nodes you wish to log into:"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/accordion.py:460
msgid "DATA"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/accordion.py:515
#, python-format
msgid ""
"You haven't created any mount points for your %(product)s %(version)s "
"installation yet.  You can:"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/accordion.py:525
msgctxt "GUI|Custom Partitioning|Autopart Page"
msgid "_Click here to create them automatically."
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/accordion.py:550
msgid "Create new mount points by clicking the '+' button."
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/accordion.py:558
msgid ""
"Or, assign new mount points to existing partitions after selecting them "
"below."
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/accordion.py:563
msgctxt "GUI|Custom Partitioning|Autopart Page"
msgid "_New mount points will use the following partitioning scheme:"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/cart.py:134
#, python-format
msgid ""
"<b>%(count)d disk; %(size)s capacity; %(free)s free space</b> (unpartitioned "
"and in file systems)"
msgid_plural ""
"<b>%(count)d disks; %(size)s capacity; %(free)s free space</b> "
"(unpartitioned and in file systems)"
msgstr[0] ""

#: pyanaconda/ui/gui/spokes/lib/cart.py:193
msgctxt "GUI|Selected Disks Dialog"
msgid "_Do not install boot loader"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/cart.py:196
#: pyanaconda/ui/gui/spokes/lib/cart.glade:183
msgctxt "GUI|Selected Disks Dialog"
msgid "_Set as Boot Device"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.py:63
#, python-format
msgid ""
"The RAID level you have selected (%(level)s) requires more disks (%(min)d) "
"than you currently have selected (%(count)d)."
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.py:67
#, python-format
msgid "CONFIGURE %(container_type)s"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.py:68
#, python-format
msgid ""
"Please create a name for this %(container_type)s and select at least one "
"disk below."
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.py:73
#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.py:74
msgid "Volume Group"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.py:73
#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.py:74
msgctxt "GUI|Custom Partitioning|Configure|Devices"
msgid "_Volume Group:"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.py:75
msgid "Volume"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.py:75
msgctxt "GUI|Custom Partitioning|Configure|Devices"
msgid "_Volume:"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.py:135
msgid "Cannot relabel already existing file system."
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.py:140
msgid "Cannot set label on file system."
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.py:142
msgid "Unacceptable label format for file system."
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.py:152
msgid "That mount point is already in use. Try something else?"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.py:154
msgid "Please enter a valid mount point."
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.py:156
#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.py:168
msgid "That mount point is invalid. Try something else?"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.py:309
msgid "container"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.py:309
msgctxt "GUI|Custom Partitioning|Configure|Devices"
msgid "container"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.py:405
#, python-format
msgid ""
"%s may be a system boot partition! Deleting it may break other operating "
"systems. Are you sure you want to delete it?"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.py:407
#, python-format
msgid "Are you sure you want to delete all of the data on %s?"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.py:409
#, python-format
msgid ""
"Are you sure you want to delete all of the data on %s, including snapshots "
"and/or subvolumes?"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.py:709
msgid "Invalid container name"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/detailederror.py:49
msgctxt "GUI|Detailed Error Dialog"
msgid "_Cancel"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/passphrase.py:38
#, python-format
msgid "You have provided a weak passphrase: %s"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/passphrase.py:39
msgid "Passphrases do not match."
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/passphrase.py:188
msgid "Passphrase contains non-ASCII characters"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/resize.py:58
msgid "Preserve"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/resize.py:59
msgid "Shrink"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/resize.py:60
msgid "Delete"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/resize.py:87
#, python-format
msgid "Installation requires a total of <b>%s</b> for system data."
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/resize.py:120
msgid "This device contains the installation source."
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/resize.py:166
#, python-format
msgid "%(freeSize)s of %(devSize)s"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/resize.py:173
msgid "Not resizeable"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/resize.py:197
msgid "Free space"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/resize.py:219
msgid ""
"You can remove existing file systems you no longer need to free up space for "
"this installation.  Removing a file system will permanently delete all of "
"the data it contains."
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/resize.py:225
msgid ""
"There is also free space available in pre-existing file systems.  While it's "
"risky and we recommend you back up your data first, you can recover that "
"free disk space and make it available for this installation below."
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/resize.py:235
#, python-format
msgid "<b>%(count)s disk; %(size)s reclaimable space</b> (in file systems)"
msgid_plural ""
"<b>%(count)s disks; %(size)s reclaimable space</b> (in file systems)"
msgstr[0] ""

#: pyanaconda/ui/gui/spokes/lib/resize.py:242
#, python-format
msgid "Total selected space to reclaim: <b>%s</b>"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/resize.py:458
#: pyanaconda/ui/gui/spokes/lib/resize.py:463
#: pyanaconda/ui/gui/spokes/lib/resize.glade:269
msgctxt "GUI|Reclaim Dialog"
msgid "Delete _all"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/resize.py:460
msgctxt "GUI|Reclaim Dialog"
msgid "Preserve _all"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/summary.py:54
#, python-format
msgid "%(description)s (%(deviceName)s)"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/summary.py:58
#, python-format
msgid "%(deviceName)s on %(container)s"
msgstr ""

#: widgets/src/BaseWindow.c:149
msgid "DISTRIBUTION INSTALLATION"
msgstr ""

#: widgets/src/BaseWindow.c:150
msgid "SPOKE NAME"
msgstr ""

#: widgets/src/BaseWindow.c:151
msgid "PRE-RELEASE / TESTING"
msgstr ""

#: widgets/src/DiskOverview.c:99
msgid "New Device"
msgstr ""

#: widgets/src/DiskOverview.c:101 widgets/src/DiskOverview.c:102
msgid "0 MB"
msgstr ""

#: widgets/src/MountpointSelector.c:86
msgid "0 GB"
msgstr ""

#: widgets/src/SpokeSelector.c:84
msgid "New Selector"
msgstr ""

#: widgets/src/SpokeWindow.c:57
msgctxt "GUI|Spoke Navigation"
msgid "_Done"
msgstr ""

#: widgets/src/StandaloneWindow.c:54
msgctxt "GUI|Standalone Navigation"
msgid "_Quit"
msgstr "Çık"

#: widgets/src/StandaloneWindow.c:55
msgctxt "GUI|Standalone Navigation"
msgid "_Continue"
msgstr "Devam Et"

#: widgets/src/LayoutIndicator.c:35
#, c-format
msgid "Current layout: '%s'. Click to switch to the next layout."
msgstr ""

#: widgets/src/LayoutIndicator.c:36
#, c-format
msgid "Current layout: '%s'. Add more layouts to enable switching."
msgstr ""

#: pyanaconda/ui/gui/spokes/advanced_user.glade:36
msgctxt "GUI|Advanced User"
msgid "_Cancel"
msgstr ""

#: pyanaconda/ui/gui/spokes/advanced_user.glade:50
msgctxt "GUI|Advanced User"
msgid "_Save Changes"
msgstr ""

#: pyanaconda/ui/gui/spokes/advanced_user.glade:82
msgid "ADVANCED USER CONFIGURATION"
msgstr ""

#: pyanaconda/ui/gui/spokes/advanced_user.glade:105
msgctxt "GUI|Advanced User"
msgid "Home _directory:"
msgstr ""

#: pyanaconda/ui/gui/spokes/advanced_user.glade:140
msgid "User and Group IDs"
msgstr ""

#: pyanaconda/ui/gui/spokes/advanced_user.glade:160
msgctxt "GUI|Advanced User"
msgid "Specify a _user ID manually:"
msgstr ""

#: pyanaconda/ui/gui/spokes/advanced_user.glade:177
msgctxt "GUI|Advanced User"
msgid "Specify a _group ID manually:"
msgstr ""

#: pyanaconda/ui/gui/spokes/advanced_user.glade:237
msgid "Group Membership"
msgstr ""

#: pyanaconda/ui/gui/spokes/advanced_user.glade:265
msgctxt "GUI|Advanced User"
msgid "_Add user to the following groups:"
msgstr ""

#: pyanaconda/ui/gui/spokes/advanced_user.glade:298
msgid "Example:"
msgstr ""

#: pyanaconda/ui/gui/spokes/advanced_user.glade:315
msgid "wheel, my-team (1245), project-x (29935)"
msgstr ""

#: pyanaconda/ui/gui/spokes/advanced_user.glade:348
msgid "Tip:"
msgstr ""

#: pyanaconda/ui/gui/spokes/advanced_user.glade:367
msgid ""
"You may input a comma-separated list of group names and group IDs here. "
"Groups that do not already exist will be created; specify their GID in "
"parentheses. "
msgstr ""

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:110
msgctxt "GUI|Date and Time|NTP"
msgid "_Cancel"
msgstr ""

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:124
msgctxt "GUI|Date and Time|NTP"
msgid "_OK"
msgstr ""

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:149
msgid "Add and mark for usage NTP servers"
msgstr ""

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:171
msgid "New NTP Server"
msgstr ""

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:190
msgid "Add NTP Server"
msgstr ""

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:210
msgid "This URL refers to a pool of NTP servers"
msgstr ""

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:256
msgid "Pool"
msgstr ""

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:269
msgid "Working"
msgstr ""

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:277
msgid "Use"
msgstr ""

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:305
#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:519
msgid "Configure NTP"
msgstr ""

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:326
#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:934
msgid "TIME & DATE"
msgstr ""

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:376
msgctxt "GUI|Date and Time"
msgid "_Region:"
msgstr ""

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:406
msgid "Region"
msgstr ""

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:423
msgctxt "GUI|Date and Time"
msgid "_City:"
msgstr ""

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:453
msgid "City"
msgstr ""

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:481
msgctxt "GUI|Date and Time"
msgid "_Network Time"
msgstr ""

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:499
msgid "Use Network Time"
msgstr ""

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:586
msgid "Hours"
msgstr ""

#. TRANSLATORS: This is the separator between hours and minutes, like in HH:MM
#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:599
msgctxt "GUI|Date and Time"
msgid ":"
msgstr ""

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:619
msgid "Minutes"
msgstr ""

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:637
msgid "Hour Up"
msgstr ""

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:655
msgid "Hour Down"
msgstr ""

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:673
msgid "Minutes Up"
msgstr ""

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:691
msgid "Minutes Down"
msgstr ""

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:735
msgid "AM/PM Up"
msgstr ""

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:771
msgid "AM/PM Down"
msgstr ""

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:822
msgctxt "GUI|Date and Time"
msgid "24-_hour"
msgstr ""

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:840
msgctxt "GUI|Date and Time"
msgid "_AM/PM"
msgstr ""

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:899
msgid "Set Date & Time"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade:60
#: pyanaconda/ui/gui/spokes/storage.glade:444
#: pyanaconda/ui/gui/spokes/storage.glade:1024
msgid "INSTALLATION DESTINATION"
msgstr "KURULUM HEDEFİ"

#: pyanaconda/ui/gui/spokes/filter.glade:129
msgctxt "GUI|Installation Destination|Filter|Search"
msgid "Search _By:"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade:146
msgid "Port / Target / LUN #"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade:147
msgid "Target WWID"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade:186
msgctxt "GUI|Installation Destination|Filter|Search|Port Target LUN"
msgid "_Port:"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade:212
msgctxt "GUI|Installation Destination|Filter|Search|Port Target LUN"
msgid "_Target:"
msgstr ""

#. check_accelerators: refresh
#: pyanaconda/ui/gui/spokes/filter.glade:241
msgctxt "GUI|Installation Destination|Filter|Search|Port Target LUN"
msgid "_LUN:"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade:283
msgctxt "GUI|Installation Destination|Filter|Search|WWID"
msgid "_WWID:"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade:333
msgctxt "GUI|Installation Destination|Filter|Search"
msgid "Search Res_ults:"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade:380
#: pyanaconda/ui/gui/spokes/filter.glade:1438
#: pyanaconda/ui/gui/spokes/source.glade:988
#: pyanaconda/ui/gui/spokes/lib/cart.glade:133
#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:235
#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:732
#: pyanaconda/ui/gui/spokes/lib/resize.glade:163
msgid "Name"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade:394
#: pyanaconda/ui/gui/spokes/filter.glade:563
#: pyanaconda/ui/gui/spokes/filter.glade:766
msgid "WWID"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade:408
#: pyanaconda/ui/gui/spokes/filter.glade:780
#: pyanaconda/ui/gui/spokes/filter.glade:1114
#: pyanaconda/ui/gui/spokes/filter.glade:1466
#: pyanaconda/ui/gui/spokes/lib/cart.glade:145
#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:248
#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:745
msgid "Capacity"
msgstr "Kapasite"

#: pyanaconda/ui/gui/spokes/filter.glade:422
#: pyanaconda/ui/gui/spokes/filter.glade:562
#: pyanaconda/ui/gui/spokes/filter.glade:808
#: pyanaconda/ui/gui/spokes/filter.glade:896
#: pyanaconda/ui/gui/spokes/filter.glade:1142
msgid "Interconnect"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade:436
msgid "Model"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade:450
#: pyanaconda/ui/gui/spokes/filter.glade:1217
#: pyanaconda/ui/gui/spokes/filter.glade:1494
msgid "LUN"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade:464
#: pyanaconda/ui/gui/spokes/filter.glade:1508
msgid "Port"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade:478
msgid "Target"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade:492
#: pyanaconda/ui/gui/spokes/filter.glade:561
#: pyanaconda/ui/gui/spokes/filter.glade:794
#: pyanaconda/ui/gui/spokes/filter.glade:895
#: pyanaconda/ui/gui/spokes/filter.glade:1128
msgid "Vendor"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade:519
msgctxt "GUI|Installation Destination|Filter"
msgid "Searc_h"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade:544
msgctxt "GUI|Installation Destination|Filter|Multipath"
msgid "Filter _By:"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade:601
msgctxt "GUI|Installation Destination|Filter|Multipath|Vendor"
msgid "Show Only _Devices From:"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade:642
msgctxt "GUI|Installation Destination|Filter|Multipath|Interconnect"
msgid "Show Only _Devices With:"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade:683
msgctxt "GUI|Installation Destination|Filter|Multipath|WWID"
msgid "Show Only _Devices Containing:"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade:822
msgid "Paths"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade:852
msgctxt "GUI|Installation Destination|Filter"
msgid "_Multipath Devices"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade:878
msgctxt "GUI|Installation Destination|Filter|Other"
msgid "Filter _By:"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade:897
msgid "ID"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade:935
msgctxt "GUI|Installation Destination|Filter|Other|Vendor"
msgid "Show Only _Devices From:"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade:976
msgctxt "GUI|Installation Destination|Filter|Other|Interconnect"
msgid "Show Only _Devices With:"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade:1017
msgctxt "GUI|Installation Destination|Filter|Other|ID"
msgid "Show Only _Devices Containing:"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade:1100
msgid "Identifier"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade:1172
msgctxt "GUI|Installation Destination|Filter"
msgid "_Other SAN Devices"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade:1198
msgctxt "GUI|Installation Destination|Filter|zSeries"
msgid "Filter B_y:"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade:1215
#: pyanaconda/ui/gui/spokes/filter.glade:1424
msgid "CCW"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade:1216
#: pyanaconda/ui/gui/spokes/filter.glade:1480
msgid "WWPN"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade:1254
msgctxt "GUI|Installation Destination|Filter|zSeries|CCW"
msgid "_CCW:"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade:1297
msgctxt "GUI|Installation Destination|Filter|zSeries|WWPN"
msgid "_WWPN:"
msgstr ""

#. check_accelerators: refresh
#: pyanaconda/ui/gui/spokes/filter.glade:1341
msgctxt "GUI|Installation Destination|Filter|zSeries|LUN"
msgid "_LUN:"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade:1452
#: pyanaconda/ui/gui/spokes/lib/summary.glade:155
msgid "Type"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade:1538
msgctxt "GUI|Installation Destination|Filter"
msgid "_zSeries Devices"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade:1561
msgctxt "GUI|Installation Destination|Filter"
msgid "_Add zFCP LUN..."
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade:1576
msgctxt "GUI|Installation Destination|Filter"
msgid "Add EC_KD DASD..."
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade:1591
msgctxt "GUI|Installation Destination|Filter"
msgid "Add _iSCSI Target..."
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade:1606
msgctxt "GUI|Installation Destination|Filter"
msgid "Add FCo_E SAN..."
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade:1621
msgctxt "GUI|Installation Destination|Filter"
msgid "Refresh _List"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade:35
msgid "Add device"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade:49
msgctxt "GUI|Network|Add Device Dialog"
msgid "_Cancel"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade:63
msgctxt "GUI|Network|Add Device Dialog"
msgid "_Add"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade:92
msgctxt "GUI|Network|Add Device Dialog"
msgid "_Select the type of device you wish to add"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade:165
#: pyanaconda/ui/gui/spokes/network.glade:2238
msgid "NETWORK & HOST NAME"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade:204
msgid ""
"Please use the live desktop environment's tools for customizing your network "
"configuration.  You can set the host name here."
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade:417
msgid "Slaves"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade:441
#: pyanaconda/ui/gui/spokes/network.glade:831
#: pyanaconda/ui/gui/spokes/network.glade:1398
msgid "DNS"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade:469
#: pyanaconda/ui/gui/spokes/network.glade:819
#: pyanaconda/ui/gui/spokes/network.glade:1386
msgid "Default Route"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade:544
#: pyanaconda/ui/gui/spokes/network.glade:783
#: pyanaconda/ui/gui/spokes/network.glade:1230
msgid "Speed"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade:569
#: pyanaconda/ui/gui/spokes/network.glade:771
msgid "Hardware Address"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade:594
msgid "Parent"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade:606
msgid "VLAN ID"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade:654
msgctxt "GUI|Network|Wired"
msgid "_Configure..."
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade:658
msgid "To apply the configuration immediately turn the device off and on."
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade:944
msgid "Security"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade:956
msgctxt "GUI|Network|Wireless"
msgid "_Network Name"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade:970
msgid "Network Name"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade:982
msgid "Security Key"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade:1066
msgctxt "GUI|Network|Wireless"
msgid "_Use as Hotspot..."
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade:1083
msgctxt "GUI|Network|Wireless"
msgid "_Stop Hotspot..."
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade:1100
msgctxt "GUI|Network|Wireless"
msgid "_Configure..."
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade:1218
msgid "IMEI"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade:1242
msgid "Provider"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade:1425
msgctxt "GUI|Network|Mobile Broadband"
msgid "_Configure..."
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade:1535
msgid "VPN Type"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade:1547
msgid "Gateway"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade:1559
msgid "Group Name"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade:1571
msgid "Group Password"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade:1583
#: pyanaconda/ui/gui/spokes/user.glade:117
msgid "User Name"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade:1691
msgctxt "GUI|Network|VPN"
msgid "_Configure..."
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade:1813
msgctxt "GUI|Network|Proxy"
msgid "_Method"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade:1838
msgctxt "GUI|Network|Proxy"
msgid "_Configuration URL"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade:1863
msgctxt "GUI|Network|Proxy"
msgid "HTTP _Proxy"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade:1877
msgctxt "GUI|Network|Proxy"
msgid "H_TTPS Proxy"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade:1891
msgctxt "GUI|Network|Proxy"
msgid "_FTP Proxy"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade:1905
msgctxt "GUI|Network|Proxy"
msgid "_Socks Host"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade:2095
msgid "Network Config Box"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade:2111
msgctxt "GUI|Network"
msgid "Unloc_k"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade:2144
msgctxt "GUI|Network"
msgid "_Airplane Mode"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade:2164
msgid "More Network Config Box"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade:2184
msgctxt "GUI|Network"
msgid "_Host Name:"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade:2244
msgid "NETWORK CONFIGURATION"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade:2289
msgid ""
"We'll need network access to fetch information about your location and to "
"make software\n"
"updates available for you."
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade:2319
msgid "Authentication"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade:2333
msgctxt "GUI|Network|Authentication Dialog"
msgid "_Cancel"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade:2348
msgctxt "GUI|Network|Authentication Dialog"
msgid "C_onnect"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade:2420
msgid ""
"Passwords or encryption keys are required to access\n"
"the wireless network"
msgstr ""

#: pyanaconda/ui/gui/spokes/software.glade:10
msgid "SOFTWARE SELECTION"
msgstr ""

#: pyanaconda/ui/gui/spokes/software.glade:66
msgid "Base Environment"
msgstr ""

#: pyanaconda/ui/gui/spokes/software.glade:84
msgid "Add-Ons for Selected Environment"
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.glade:27
msgctxt "GUI|Storage|Need Space Dialog"
msgid "_Cancel & add more disks"
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.glade:43
msgctxt "GUI|Storage|Need Space Dialog"
msgid "_Reclaim space"
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.glade:78
#: pyanaconda/ui/gui/spokes/storage.glade:293
msgid "INSTALLATION OPTIONS"
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.glade:95
#: pyanaconda/ui/gui/spokes/storage.glade:309
msgid ""
"Here we'll describe how much space is needed for the current software "
"selection."
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.glade:121
#: pyanaconda/ui/gui/spokes/storage.glade:335
msgid "disk free"
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.glade:137
#: pyanaconda/ui/gui/spokes/storage.glade:350
msgid "Free space available for use."
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.glade:153
#: pyanaconda/ui/gui/spokes/storage.glade:366
msgid "fs free"
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.glade:169
#: pyanaconda/ui/gui/spokes/storage.glade:382
msgid "Free space unavailable but reclaimable from existing partitions."
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.glade:192
#: pyanaconda/ui/gui/spokes/storage.glade:405
msgid "Here we'll describe what your options are."
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.glade:219
msgid "Need Space"
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.glade:242
msgctxt "GUI|Storage|No Space Dialog"
msgid "_Quit installer"
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.glade:258
msgctxt "GUI|Storage|No Space Dialog"
msgid "_Cancel"
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.glade:432
msgid "No Space"
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.glade:507
msgid "Device Selection"
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.glade:526
msgid ""
"Select the device(s) you'd like to install to.  They will be left untouched "
"until you click on the main menu's \"Begin Installation\" button."
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.glade:544
msgid "Local Standard Disks"
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.glade:608
#: pyanaconda/ui/gui/spokes/storage.glade:748
msgid "Disks left unselected here will not be touched."
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.glade:627
msgid "Specialized & Network Disks"
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.glade:697
msgctxt "GUI|Storage"
msgid "_Add a disk..."
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.glade:714
msgid "Add Specialized Disk"
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.glade:767
msgid "Other Storage Options"
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.glade:793
msgid "Partitioning"
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.glade:810
msgid "Encryption"
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.glade:823
msgctxt "GUI|Storage"
msgid "A_utomatically configure partitioning."
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.glade:839
msgctxt "GUI|Storage"
msgid "_I will configure partitioning."
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.glade:871
msgctxt "GUI|Storage"
msgid "_Encrypt my data."
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.glade:886
msgid "You'll set a passphrase next."
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.glade:907
msgctxt "GUI|Storage"
msgid "I would like to _make additional space available."
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.glade:923
msgid "Storage Options"
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.glade:955
msgid "summary"
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.glade:988
msgctxt "GUI|Storage"
msgid "_Full disk summary and boot loader..."
msgstr ""

#: pyanaconda/ui/gui/spokes/keyboard.glade:15
#: pyanaconda/ui/gui/spokes/keyboard.glade:388
msgid "KEYBOARD LAYOUT"
msgstr ""

#: pyanaconda/ui/gui/spokes/keyboard.glade:56
msgid ""
"Which keyboard layouts would you like to use on this system?  You may move "
"any layout to the top of the list to select it as the default."
msgstr ""

#: pyanaconda/ui/gui/spokes/keyboard.glade:96
msgid ""
"Changes here will only apply to the installed system. Use the desktop's tool "
"to configure the keyboard for the installation process."
msgstr ""

#: pyanaconda/ui/gui/spokes/keyboard.glade:167
msgid "Selected Layouts"
msgstr ""

#: pyanaconda/ui/gui/spokes/keyboard.glade:190
msgctxt "GUI|Keyboard Layout"
msgid "_Add layout"
msgstr ""

#: pyanaconda/ui/gui/spokes/keyboard.glade:205
msgctxt "GUI|Keyboard Layout"
msgid "_Remove layout"
msgstr ""

#: pyanaconda/ui/gui/spokes/keyboard.glade:220
msgctxt "GUI|Keyboard Layout"
msgid "Move selected layout _up"
msgstr ""

#: pyanaconda/ui/gui/spokes/keyboard.glade:235
msgctxt "GUI|Keyboard Layout"
msgid "Move selected layout _down"
msgstr ""

#: pyanaconda/ui/gui/spokes/keyboard.glade:251
msgctxt "GUI|Keyboard Layout"
msgid "_Preview layout"
msgstr ""

#: pyanaconda/ui/gui/spokes/keyboard.glade:291
msgctxt "GUI|Keyboard Layout"
msgid "_Test the layout configuration below:"
msgstr ""

#: pyanaconda/ui/gui/spokes/keyboard.glade:331
msgid "Alt + Shift to switch layouts."
msgstr ""

#: pyanaconda/ui/gui/spokes/keyboard.glade:346
msgctxt "GUI|Keyboard Layout"
msgid "_Options"
msgstr ""

#: pyanaconda/ui/gui/spokes/keyboard.glade:421
msgctxt "GUI|Keyboard Layout|Add Layout"
msgid "_Cancel"
msgstr ""

#: pyanaconda/ui/gui/spokes/keyboard.glade:435
msgctxt "GUI|Keyboard Layout|Add Layout"
msgid "_Add"
msgstr ""

#: pyanaconda/ui/gui/spokes/keyboard.glade:467
msgid "ADD A KEYBOARD LAYOUT"
msgstr ""

#: pyanaconda/ui/gui/spokes/keyboard.glade:483
msgid "You may add a keyboard layout by selecting it below:"
msgstr ""

#: pyanaconda/ui/gui/spokes/keyboard.glade:532
msgid "Available Layouts"
msgstr ""

#: pyanaconda/ui/gui/spokes/keyboard.glade:573
msgid "Add Layout"
msgstr ""

#: pyanaconda/ui/gui/spokes/keyboard.glade:605
msgctxt "GUI|Keyboard Layout|Switching Options"
msgid "_Cancel"
msgstr ""

#: pyanaconda/ui/gui/spokes/keyboard.glade:619
msgctxt "GUI|Keyboard Layout|Switching Options"
msgid "_OK"
msgstr ""

#: pyanaconda/ui/gui/spokes/keyboard.glade:651
msgid "LAYOUT SWITCHING OPTIONS"
msgstr ""

#: pyanaconda/ui/gui/spokes/keyboard.glade:668
msgid ""
"Which combination(s) would you prefer for switching between keyboard layouts?"
msgstr ""

#: pyanaconda/ui/gui/spokes/keyboard.glade:751
msgid "Layout Options"
msgstr ""

#: pyanaconda/ui/gui/spokes/langsupport.glade:33
msgid "LANGUAGE SUPPORT"
msgstr ""

#: pyanaconda/ui/gui/spokes/langsupport.glade:78
msgid "Select additional language support to be installed:"
msgstr ""

#: pyanaconda/ui/gui/spokes/langsupport.glade:186
#: pyanaconda/ui/gui/spokes/welcome.glade:329
msgid "Type here to search."
msgstr ""

#: pyanaconda/ui/gui/spokes/password.glade:8
#: pyanaconda/ui/gui/spokes/password.glade:191
msgid "ROOT PASSWORD"
msgstr ""

#: pyanaconda/ui/gui/spokes/password.glade:53
msgctxt "GUI|Password"
msgid "_Root Password:"
msgstr ""

#: pyanaconda/ui/gui/spokes/password.glade:67
msgctxt "GUI|Password"
msgid "_Confirm:"
msgstr ""

#: pyanaconda/ui/gui/spokes/password.glade:104
#: pyanaconda/ui/gui/spokes/user.glade:188
msgid "Confirm Password"
msgstr ""

#: pyanaconda/ui/gui/spokes/password.glade:138
#: pyanaconda/ui/gui/spokes/user.glade:251
msgid "empty password"
msgstr ""

#: pyanaconda/ui/gui/spokes/password.glade:159
msgid ""
"The root account is used for administering the system.  Enter a password for "
"the root user."
msgstr ""

#: pyanaconda/ui/gui/spokes/source.glade:29
msgctxt "GUI|Software Source|ISO Chooser Dialog"
msgid "_Cancel"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.glade:43
msgctxt "GUI|Software Source|ISO Chooser Dialog"
msgid "_Open"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.glade:95
msgctxt "GUI|Software Source|Media Check Dialog"
msgid "_Done"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.glade:128
msgid "MEDIA VERIFICATION"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.glade:155
msgid "Verifying media, please wait..."
msgstr ""

#: pyanaconda/ui/gui/spokes/source.glade:224
msgctxt "GUI|Software Source|Proxy Dialog"
msgid "_Cancel"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.glade:238
msgctxt "GUI|Software Source|Proxy Dialog"
msgid "_OK"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.glade:267
msgctxt "GUI|Software Source|Proxy Dialog"
msgid "_Enable HTTP Proxy"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.glade:299
msgctxt "GUI|Software Source|Proxy Dialog"
msgid "_Proxy Host:"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.glade:316
msgid "<span size=\"small\"><b>Example:</b> squid.mysite.org:3128</span>"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.glade:349
msgctxt "GUI|Software Source|Proxy Dialog"
msgid "_Use Authentication"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.glade:375
msgctxt "GUI|Software Source|Proxy Dialog"
msgid "User _name:"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.glade:392
msgctxt "GUI|Software Source|Proxy Dialog"
msgid "Pass_word:"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.glade:474
msgid "INSTALLATION SOURCE"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.glade:531
msgid "Which installation source would you like to use?"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.glade:545
msgctxt "GUI|Software Source"
msgid "_Auto-detected installation media:"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.glade:571
msgid "Device:"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.glade:583
msgid "Label:"
msgstr ""

#. check_accelerators: verifyMediaButton
#: pyanaconda/ui/gui/spokes/source.glade:593
#: pyanaconda/ui/gui/spokes/source.glade:693
msgctxt "GUI|Software Source"
msgid "_Verify"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.glade:617
msgctxt "GUI|Software Source"
msgid "_ISO file:"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.glade:645
msgctxt "GUI|Software Source"
msgid "D_evice:"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.glade:676
msgctxt "GUI|Software Source"
msgid "_Choose an ISO"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.glade:717
msgctxt "GUI|Software Source"
msgid "_On the network:"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.glade:781
msgctxt "GUI|Software Source"
msgid "_Proxy setup..."
msgstr ""

#: pyanaconda/ui/gui/spokes/source.glade:796
msgctxt "GUI|Software Source"
msgid "This URL refers to a _mirror list."
msgstr ""

#: pyanaconda/ui/gui/spokes/source.glade:834
msgctxt "GUI|Software Source"
msgid "N_FS mount options:"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.glade:849
#: pyanaconda/ui/gui/spokes/source.glade:850
msgid "This field is optional."
msgstr ""

#: pyanaconda/ui/gui/spokes/source.glade:883
msgid "Updates"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.glade:910
msgctxt "GUI|Software Source"
msgid ""
"Don't install the latest available software _updates. Install the default "
"versions provided by the installation source above."
msgstr ""

#: pyanaconda/ui/gui/spokes/source.glade:934
msgid "Additional repositories"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.glade:975
msgid "Enabled"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.glade:1020
#: pyanaconda/ui/gui/spokes/source.glade:1021
msgid "Add a new repository."
msgstr ""

#: pyanaconda/ui/gui/spokes/source.glade:1022
msgctxt "GUI|Software Source"
msgid "A_dd"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.glade:1039
#: pyanaconda/ui/gui/spokes/source.glade:1040
msgid "Remove the selected repository."
msgstr ""

#: pyanaconda/ui/gui/spokes/source.glade:1041
msgctxt "GUI|Software Source"
msgid "_Remove"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.glade:1058
#: pyanaconda/ui/gui/spokes/source.glade:1059
msgid "Revert to the previous list of repositories."
msgstr ""

#: pyanaconda/ui/gui/spokes/source.glade:1060
msgctxt "GUI|Software Source"
msgid "Rese_t"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.glade:1090
#: pyanaconda/ui/gui/spokes/source.glade:1091
msgid "URL for the repository, without protocol."
msgstr ""

#: pyanaconda/ui/gui/spokes/source.glade:1103
#: pyanaconda/ui/gui/spokes/source.glade:1104
msgid "URL of proxy in the form of protocol://host:[port]"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.glade:1116
#: pyanaconda/ui/gui/spokes/source.glade:1117
msgid "Optional proxy user name."
msgstr ""

#: pyanaconda/ui/gui/spokes/source.glade:1129
msgid "Optional proxy password."
msgstr ""

#: pyanaconda/ui/gui/spokes/source.glade:1141
msgctxt "GUI|Software Source"
msgid "This URL refers to a mirror _list."
msgstr ""

#: pyanaconda/ui/gui/spokes/source.glade:1160
msgctxt "GUI|Software Source"
msgid "_Name:"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.glade:1177
#: pyanaconda/ui/gui/spokes/source.glade:1178
msgid "Protocol for the repository URL."
msgstr ""

#: pyanaconda/ui/gui/spokes/source.glade:1197
msgctxt "GUI|Software Source"
msgid "Pro_xy URL:"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.glade:1214
msgctxt "GUI|Software Source"
msgid "U_ser name:"
msgstr "Kullanıcı Adı"

#: pyanaconda/ui/gui/spokes/source.glade:1231
msgctxt "GUI|Software Source"
msgid "Pass_word:"
msgstr "Parola"

#: pyanaconda/ui/gui/spokes/source.glade:1248
#: pyanaconda/ui/gui/spokes/source.glade:1249
msgid "Name of the repository."
msgstr ""

#: pyanaconda/ui/gui/spokes/welcome.glade:23
msgctxt "GUI|Welcome|Beta Warn Dialog"
msgid "_Get me out of here!"
msgstr ""

#: pyanaconda/ui/gui/spokes/welcome.glade:37
msgctxt "GUI|Welcome|Beta Warn Dialog"
msgid "_I accept my fate."
msgstr ""

#: pyanaconda/ui/gui/spokes/welcome.glade:68
msgid "This is unstable, pre-release software."
msgstr ""

#: pyanaconda/ui/gui/spokes/welcome.glade:97
msgid ""
"You wake up inside an OS installer in Timbuktu, and it's six months in the "
"future.  But, there are bugs.  Bugs everywhere.  Bugs you must live with.  "
"This OS of the future isn't a stable OS you can rely on.  It's for testing "
"purposes only."
msgstr ""

#: pyanaconda/ui/gui/spokes/welcome.glade:126
msgid "Beta Warn"
msgstr ""

#: pyanaconda/ui/gui/spokes/welcome.glade:224
msgid "What language would you like to use during the installation process?"
msgstr "Kurulum sırasında hangi dili kullanmak istiyorsunuz?"

#: pyanaconda/ui/gui/spokes/welcome.glade:309
msgid "Languages"
msgstr "Diller"

#: pyanaconda/ui/gui/spokes/welcome.glade:381
msgid "Locales"
msgstr "Yerler"

#: pyanaconda/ui/gui/spokes/welcome.glade:415
msgid "WELCOME"
msgstr "HOŞGELDİNİZ"

#: pyanaconda/ui/gui/spokes/welcome.glade:436
msgctxt "GUI|Welcome|Unsupported Hardware Dialog"
msgid "_Quit"
msgstr "Çık"

#: pyanaconda/ui/gui/spokes/welcome.glade:450
msgctxt "GUI|Welcome|Unsupported Hardware Dialog"
msgid "_Continue"
msgstr "Devam Et"

#: pyanaconda/ui/gui/spokes/welcome.glade:481
msgid "Unsupported Hardware Detected"
msgstr ""

#: pyanaconda/ui/gui/spokes/welcome.glade:539
msgid "Unsupported Hardware"
msgstr ""

#: pyanaconda/ui/gui/spokes/user.glade:10
#: pyanaconda/ui/gui/spokes/user.glade:347
msgid "CREATE USER"
msgstr "KULLANICI OLUŞTUR"

#: pyanaconda/ui/gui/spokes/user.glade:62
msgctxt "GUI|User"
msgid "_Full name"
msgstr "Tam Adı"

#: pyanaconda/ui/gui/spokes/user.glade:80
msgctxt "GUI|User"
msgid "_User name"
msgstr "Kullanıcı Adı"

#: pyanaconda/ui/gui/spokes/user.glade:100
msgid "Full Name"
msgstr "Tam Adı"

#: pyanaconda/ui/gui/spokes/user.glade:132
msgctxt "GUI|User"
msgid "_Password"
msgstr "Parola"

#: pyanaconda/ui/gui/spokes/user.glade:150
msgctxt "GUI|User"
msgid "_Confirm password"
msgstr "Parolayı Onayla"

#: pyanaconda/ui/gui/spokes/user.glade:202
msgid ""
"<b>Tip:</b> Keep your user name shorter than 32 characters and do not use "
"spaces."
msgstr ""

#: pyanaconda/ui/gui/spokes/user.glade:212
msgctxt "GUI|User"
msgid "_Require a password to use this account"
msgstr ""

#: pyanaconda/ui/gui/spokes/user.glade:270
msgctxt "GUI|User"
msgid "_Make this user administrator"
msgstr ""

#: pyanaconda/ui/gui/spokes/user.glade:290
msgctxt "GUI|User"
msgid "_Advanced..."
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.glade:68
#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:543
msgid "RAID0"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.glade:72
#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:547
msgid "RAID1"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.glade:76
#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:551
msgid "RAID4"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.glade:80
#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:555
msgid "RAID5"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.glade:84
#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:559
msgid "RAID6"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.glade:88
#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:563
msgid "RAID10"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.glade:187
msgid "Add a new mount point."
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.glade:188
msgctxt "GUI|Custom Partitioning"
msgid "Add"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.glade:203
msgid "Remove the selected mount point(s)."
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.glade:204
msgctxt "GUI|Custom Partitioning"
msgid "Remove"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.glade:219
msgid "Reload storage configuration from disk."
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.glade:220
msgctxt "GUI|Custom Partitioning"
msgid "Refresh"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.glade:285
#: pyanaconda/ui/gui/spokes/custom.glade:1026
#: pyanaconda/ui/gui/spokes/custom.glade:1171
#: pyanaconda/ui/gui/spokes/custom.glade:1258
msgid "Selected Device"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.glade:303
#: pyanaconda/ui/gui/spokes/custom.glade:1044
#: pyanaconda/ui/gui/spokes/custom.glade:1189
#: pyanaconda/ui/gui/spokes/custom.glade:1276
msgid "Device description"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.glade:364
msgctxt "GUI|Custom Partitioning|Configure|Devices"
msgid "Mount _Point:"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.glade:411
msgctxt "GUI|Custom Partitioning|Configure|Devices"
msgid "_Desired Capacity:"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.glade:462
msgctxt "GUI|Custom Partitioning|Configure|Devices"
msgid "_Label:"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.glade:509
msgid "Device(s):"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.glade:538
msgctxt "GUI|Custom Partitioning|Configure|Devices"
msgid "_Modify..."
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.glade:570
msgctxt "GUI|Custom Partitioning|Configure|Devices"
msgid "_Name:"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.glade:625
msgctxt "GUI|Custom Partitioning|Configure"
msgid "Device _Type:"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.glade:669
msgctxt "GUI|Custom Partitioning|Configure"
msgid "_Encrypt"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.glade:710
msgctxt "GUI|Custom Partitioning|Configure"
msgid "File S_ystem:"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.glade:754
msgctxt "GUI|Custom Partitioning|Configure"
msgid "Ref_ormat"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.glade:808
msgctxt "GUI|Custom Partitioning|Configure"
msgid "_Volume Group:"
msgstr ""

#. check_accelerators: configureButton
#: pyanaconda/ui/gui/spokes/custom.glade:861
msgctxt "GUI|Custom Partitioning|Configure"
msgid "_Modify..."
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.glade:893
msgctxt "GUI|Custom Partitioning|Configure"
msgid "RA_ID Level:"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.glade:958
msgctxt "GUI|Custom Partitioning|Configure|Devices"
msgid "_Update Settings"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.glade:981
msgid ""
"Note:  The settings you make on this screen will not be applied until you "
"click on the main menu's 'Begin Installation' button."
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.glade:1078
msgid ""
"This device is encrypted and cannot be read without a valid passphrase. You "
"may unlock it below."
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.glade:1099
msgctxt "GUI|Custom Partitioning|Encrypted"
msgid "_Passphrase:"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.glade:1128
msgctxt "GUI|Custom Partitioning|Encrypted"
msgid "_Unlock"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.glade:1223
msgid ""
"This device cannot be edited directly. You can remove it or select a "
"different device."
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.glade:1371
msgid "AVAILABLE SPACE"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.glade:1424
msgid "TOTAL SPACE"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.glade:1464
msgctxt "GUI|Custom Partitioning"
msgid "_Reset All"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/fcoe.glade:23
msgctxt "GUI|Advanced Storage|FCoE"
msgid "_Cancel"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/fcoe.glade:56
msgid ""
"Please select the network interface which is connected to your FCoE switch."
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/fcoe.glade:70
msgctxt "GUI|Advanced Storage|FCoE"
msgid "_NIC:"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/fcoe.glade:92
msgctxt "GUI|Advanced Storage|FCoE"
msgid "Use _DCB"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/fcoe.glade:108
msgctxt "GUI|Advanced Storage|FCoE"
msgid "Use auto _VLAN"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/fcoe.glade:144
msgctxt "GUI|Advanced Storage|FCoE"
msgid "_Add FCoE Disk(s)"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/fcoe.glade:187
msgid "Error adding FCoE SAN."
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/fcoe.glade:215
msgid ""
"The following error occurred while attempting to add an FCoE SAN.  Please "
"check your configuration and try again"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:40
msgctxt "GUI|Advanced Storage|iSCSI|Configure"
msgid "_Cancel"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:54
msgctxt "GUI|Advanced Storage|iSCSI|Configure"
msgid "_OK"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:85
msgid "ADD iSCSI STORAGE TARGET"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:114
msgid ""
"To use iSCSI disks, you must provide the address of your iSCSI target and "
"the iSCSI initiator name you've configured for your host."
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:142
msgctxt "GUI|Advanced Storage|iSCSI|Configure"
msgid "_Target IP Address:"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:170
msgctxt "GUI|Advanced Storage|iSCSI|Configure"
msgid "iSCSI _Initiator Name:"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:187
#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:760
msgid "No credentials (discovery authentication disabled)"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:188
#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:761
msgid "CHAP pair"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:189
#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:762
msgid "CHAP pair and a reverse pair"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:203
msgctxt "GUI|Advanced Storage|iSCSI|Configure"
msgid "_Discovery Authentication Type:"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:218
msgid ""
"<span size=\"small\"><b>Example:</b> iqn.2012-09.com.example:diskarrays-sn-"
"a8675309</span>"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:263
msgctxt "GUI|Advanced Storage|iSCSI|Configure|CHAP"
msgid "CHAP _User name:"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:277
msgctxt "GUI|Advanced Storage|iSCSI|Configure|CHAP"
msgid "CHAP _Password:"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:389
msgctxt "GUI|Advanced Storage|iSCSI|Configure|Reverse CHAP"
msgid "CHAP _User name:"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:403
msgctxt "GUI|Advanced Storage|iSCSI|Configure|Reverse CHAP"
msgid "CHAP _Password:"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:417
msgctxt "GUI|Advanced Storage|iSCSI|Configure|Reverse CHAP"
msgid "Reverse CHAP User _name:"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:431
msgctxt "GUI|Advanced Storage|iSCSI|Configure|Reverse CHAP"
msgid "Reverse CHAP Pass_word:"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:465
msgctxt "GUI|Advanced Storage|iSCSI|Configure|Start"
msgid "_Start Discovery"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:502
msgid "Discovering iSCSI targets.  This may take a moment..."
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:541
msgid "Discovery login rejected."
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:556
msgid ""
"The following error occurred discovering iSCSI targets.  Please double check "
"your authorization information and try again."
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:579
msgctxt "GUI|Advanced Storage|iSCSI|Configure|Retry"
msgid "_Retry Discovery"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:619
msgctxt "GUI|Advanced Storage|iSCSI|Configure"
msgid "_Bind targets to network interfaces This may take a moment..."
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:695
msgid "Node Name"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:706
msgid "Interface"
msgstr "Arayüz"

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:717
msgid "Portal"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:745
msgctxt "GUI|Advanced Storage|iSCSI|Login"
msgid "_Node Login Authentication Type:"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:763
msgid "Use the credentials from discovery"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:797
msgctxt "GUI|Advanced Storage|iSCSI|Login|CHAP"
msgid "CHAP _User name:"
msgstr "Kullanıcı Adı"

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:811
msgctxt "GUI|Advanced Storage|iSCSI|Login|CHAP"
msgid "CHAP _Password:"
msgstr "Parola"

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:923
msgctxt "GUI|Advanced Storage|iSCSI|Login|Reverse CHAP"
msgid "CHAP _User name:"
msgstr "Kullanıcı Adı"

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:937
msgctxt "GUI|Advanced Storage|iSCSI|Login|Reverse CHAP"
msgid "CHAP _Password:"
msgstr "Parola"

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:951
msgctxt "GUI|Advanced Storage|iSCSI|Login|Reverse CHAP"
msgid "_Reverse CHAP User name:"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:965
msgctxt "GUI|Advanced Storage|iSCSI|Login|Reverse CHAP"
msgid "Reverse CHAP Pass_word:"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:1014
msgctxt "GUI|Advanced Storage|iSCSI|Login|Login"
msgid "_Log In"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:1069
msgid "Node login failed."
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:1084
msgid ""
"The following error occurred logging into the selected iSCSI node.  Please "
"double check your authorization information and try again"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:1107
msgctxt "GUI|Advanced Storage|iSCSI|Login|Retry"
msgid "Retry _Log In"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/dasd.glade:22
msgctxt "GUI|Advanced Storage|DASD|Configure"
msgid "_Cancel"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/dasd.glade:36
msgctxt "GUI|Advanced Storage|DASD|Configure"
msgid "_OK"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/dasd.glade:67
msgid "ADD DASD STORAGE TARGET"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/dasd.glade:90
msgid "To use DASD disks, you must provide the device number."
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/dasd.glade:117
msgctxt "GUI|Advanced Storage|DASD|Configure"
msgid "_Device number:"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/dasd.glade:136
msgctxt "GUI|Advanced Storage|DASD|Configure"
msgid "_Start Discovery"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/dasd.glade:173
msgid "Discovering DASD devices.  This may take a moment..."
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/dasd.glade:212
#: pyanaconda/ui/gui/spokes/advstorage/zfcp.glade:253
msgid "Device discovery failed."
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/dasd.glade:227
msgid ""
"The following error occurred discovering DASD devices.  Please double check "
"your configuration information and try again."
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/dasd.glade:250
msgctxt "GUI|Advanced Storage|DASD|Configure"
msgid "_Retry Discovery"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/dasd.glade:307
msgid "Device discovery succeeded."
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/dasd.glade:324
msgid "The added DASD can be found under \"Local Standard Disks.\""
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/zfcp.glade:82
msgid "ADD zFCP STORAGE TARGET"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/zfcp.glade:105
msgid ""
"To use zFCP disks, you must provide the device number, WWPN, and LUN "
"configured for the device."
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/zfcp.glade:132
msgctxt "GUI|Advanced Storage|zFCP|Device Number"
msgid "_Device number:"
msgstr "Aygıt numarası:"

#: pyanaconda/ui/gui/spokes/advstorage/zfcp.glade:159
msgid "WWPN:"
msgstr "WWPN:"

#: pyanaconda/ui/gui/spokes/advstorage/zfcp.glade:177
msgctxt "GUI|Advanced Storage|zFCP|Start Discovery"
msgid "_Start Discovery"
msgstr "Tanımlamayı Başlat"

#: pyanaconda/ui/gui/spokes/advstorage/zfcp.glade:214
msgid "Discovering zFCP devices.  This may take a moment..."
msgstr "zFCP aygıtı tanımlanıyor. Bu biraz zaman alabilir..."

#: pyanaconda/ui/gui/spokes/advstorage/zfcp.glade:268
msgid ""
"The following error occurred discovering zFCP devices.  Please double check "
"your configuration information and try again."
msgstr ""
"Aşağıdaki hata zFCP aygıtnı tanımladıkdan sonra oluştu. Lütfen yapılandırma "
"bilgilerini girin ve yeniden deneyin."

#: pyanaconda/ui/gui/spokes/advstorage/zfcp.glade:291
msgctxt "GUI|Advanced Storage|zFCP|Retry Discovery"
msgid "_Retry Discovery"
msgstr "Yeniden tanımla"

#: pyanaconda/ui/gui/spokes/advstorage/zfcp.glade:333
msgid "LUN:"
msgstr "LUN:"

#: pyanaconda/ui/gui/spokes/lib/cart.glade:26
#: pyanaconda/ui/gui/spokes/lib/cart.glade:78
msgid "SELECTED DISKS"
msgstr "SEÇİLEN DİSKLER"

#: pyanaconda/ui/gui/spokes/lib/cart.glade:43
msgctxt "GUI|Selected Disks Dialog"
msgid "_Close"
msgstr "Kapat"

#: pyanaconda/ui/gui/spokes/lib/cart.glade:106
msgid "Boot"
msgstr "Önyükle"

#: pyanaconda/ui/gui/spokes/lib/cart.glade:120
#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:222
#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:719
msgid "Description"
msgstr "Açıklama"

#: pyanaconda/ui/gui/spokes/lib/cart.glade:158
#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:261
#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:758
msgid "Free"
msgstr "Boş"

#: pyanaconda/ui/gui/spokes/lib/cart.glade:199
msgctxt "GUI|Selected Disks Dialog"
msgid "_Remove"
msgstr "Sil"

#: pyanaconda/ui/gui/spokes/lib/cart.glade:226
msgid "Disk summary goes here"
msgstr "Disk özeti burada"

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:21
msgctxt "GUI|Custom Partitioning|Confirm Delete Dialog"
msgid "_Cancel"
msgstr "İptal"

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:37
msgctxt "GUI|Custom Partitioning|Confirm Delete Dialog"
msgid "_Delete It"
msgstr "Sil"

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:129
msgctxt "GUI|Custom Partitioning|Configure Dialog"
msgid "_Cancel"
msgstr "İptal"

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:144
msgctxt "GUI|Custom Partitioning|Configure Dialog"
msgid "_Select"
msgstr "Seç"

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:175
msgid "CONFIGURE MOUNT POINT"
msgstr "BAĞLAMA NOKTASI AYARLARI"

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:191
msgid "Select one or more disks this device may reside on."
msgstr "Bu aygıtın bulunabileceği bir veya daha fazla disk seçin."

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:321
msgctxt "GUI|Custom Partitioning|Add Dialog"
msgid "_Cancel"
msgstr "İptal"

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:336
msgctxt "GUI|Custom Partitioning|Add Dialog"
msgid "_Add mount point"
msgstr "Bağlama noktası ekle"

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:370
msgctxt "GUI|Custom Partitioning|Add Dialog"
msgid "_Desired Capacity:"
msgstr "İstenen Kapasite:"

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:387
msgctxt "GUI|Custom Partitioning|Add Dialog"
msgid "Mount _Point:"
msgstr "Bağlama Noktası"

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:405
msgid "ADD A NEW MOUNT POINT"
msgstr "YENİ BAĞLAMA NOKTASI EKLE"

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:423
msgid ""
"More customization options are available\n"
"after creating the mount point below."
msgstr ""
"Daha fazla özelleştirme seçenekleri mevcut.\n"
"Bölümü oluşturduktan sonra kullanabilirsiniz."

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:458
msgid ""
"That mount point is already in\n"
"use.  Try something else?"
msgstr ""
"Bu bağlama noktası zaten kullanılıyor. \n"
"Başka birini deneyelim mi?"

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:482
msgid "eg: \"20 GB\", \"500mb\" (minus the quotation marks)"
msgstr "örn: \"20 GB\", \"500mb\" (tırnak işaretleri olmadan)"

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:539
msgid "Single"
msgstr "Tek"

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:587
msgctxt "GUI|Custom Partitioning|Container Dialog"
msgid "_Cancel"
msgstr "İptal"

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:601
msgctxt "GUI|Custom Partitioning|Container Dialog"
msgid "_Save"
msgstr "Kaydet"

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:633
msgid "CONFIGURE CONTAINER"
msgstr "YAPILANDIRMA AYARLARI"

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:649
msgid ""
"Please create a name for this container and select at least one disk below."
msgstr ""
"Bu yapılandırma için bir ad oluşturun ve aşağıdan bir disk seçin."

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:667
msgctxt "GUI|Custom Partitioning|Container Dialog"
msgid "_Name:"
msgstr "Ad"

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:786
msgid "RAID Level:"
msgstr "RAID Seviyesi"

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:815
msgid "Encrypt"
msgstr "Şifrele"

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:856
msgid "Please enter a valid name."
msgstr "Lütfen geçerli bir isim girin."

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:886
msgctxt "GUI|Custom Partitioning|Container Dialog"
msgid "Si_ze policy:"
msgstr "Boyut politikası:"

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:901
msgid "Automatic"
msgstr "Otomatik"

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:902
msgid "As large as possible"
msgstr "Mümkün olduğunca büyük"

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:903
msgid "Fixed"
msgstr "Sabit"

#: pyanaconda/ui/gui/spokes/lib/dasdfmt.glade:8
msgid "UNFORMATTED DASDS"
msgstr "BİÇİMLENDİRİLMEMİŞ DASDS'LER"

#: pyanaconda/ui/gui/spokes/lib/dasdfmt.glade:23
msgctxt "GUI|DAD Format Dialog"
msgid "_Cancel"
msgstr "İptal"

#: pyanaconda/ui/gui/spokes/lib/dasdfmt.glade:37
msgctxt "GUI|DAD Format Dialog"
msgid "_OK"
msgstr "TAMAM"

#: pyanaconda/ui/gui/spokes/lib/dasdfmt.glade:70
msgid ""
"The following unformatted DASDs have been detected on your system. You can "
"choose to format them now with dasdfmt or cancel to leave them unformatted. "
"Unformatted DASDs cannot be used during installation."
msgstr ""
"Sisteminizde biçimlendirilmemiş DASD'ler tespit edildi. dasdfmt kullanarak "
"şimdi biçimlendirebilir veya iptal'i seçerek biçimlendirilmemiş olarak "
"bırakabilirsiniz. Biçimlendirilmeyen diskler, yükleme sırasında kullanılamaz."

#: pyanaconda/ui/gui/spokes/lib/dasdfmt.glade:86
msgid "Unformatted DASDs Detected"
msgstr "Biçimlendirilmemiş DASD'ler algılandı "

#: pyanaconda/ui/gui/spokes/lib/dasdfmt.glade:136
msgid ""
"<b>Warning</b>: All storage changes made using the installer will be lost "
"when you choose to format."
msgstr ""
"<b>Uyarı</b>: Eğer biçimlendir seçeneğini seçerseniz, "
"yapılandırdığınız tüm depolama değişiklikleri kaybolacak."

#: pyanaconda/ui/gui/spokes/lib/dasdfmt.glade:160
msgctxt "GUI|DAD Format Dialog"
msgid "_Format with dasdfmt"
msgstr "dasdfmt kullanarak biçimlendir"

#: pyanaconda/ui/gui/spokes/lib/dasdfmt.glade:207
msgid ""
"You may <a href=\"\">go back to the main menu</a> to complete other\n"
"installation options while formatting completes."
msgstr ""
"Biçimlendirme tamamlanırken diğer yükleme seçeneklerini tamamlamak\n"
"için <a href=\"\">ana menü</a>'ye geri dönebilirsiniz."

#: pyanaconda/ui/gui/spokes/lib/dasdfmt.glade:247
msgid "Disk formatting complete."
msgstr "Disk Biçimlendirme Tamamlandı."

#: pyanaconda/ui/gui/spokes/lib/detailederror.glade:25
msgid "An unknown error occurred during installation.  Details are below."
msgstr "Kurulum sırasında bilinmeyen bir hata oluştu. Detaylar aşağıdaki gibidir."

#: pyanaconda/ui/gui/spokes/lib/passphrase.glade:23
msgctxt "GUI|Passphrase Dialog"
msgid "_Cancel"
msgstr "İptal"

#: pyanaconda/ui/gui/spokes/lib/passphrase.glade:38
msgctxt "GUI|Passphrase Dialog"
msgid "_Save Passphrase"
msgstr "Parolayı Kaydet"

#: pyanaconda/ui/gui/spokes/lib/passphrase.glade:65
msgid "DISK ENCRYPTION PASSPHRASE"
msgstr "DİSK ŞİFRELEME PAROLASI"

#: pyanaconda/ui/gui/spokes/lib/passphrase.glade:81
msgid ""
"You have chosen to encrypt some of your data. You will need to create a "
"passphrase that you will use to access your data when you start your "
"computer."
msgstr ""
"Bazı verilerin şifrelenmesini seçtiniz. Bilgisayarınızı yeniden başlattığınızda "
"verilerinize erişmek için kullanılacak bir parola oluşturmanız gerekli."

#: pyanaconda/ui/gui/spokes/lib/passphrase.glade:102
msgctxt "GUI|Passphrase Dialog"
msgid "_Passphrase:"
msgstr "Parola"

#: pyanaconda/ui/gui/spokes/lib/passphrase.glade:132
msgctxt "GUI|Passphrase Dialog"
msgid "Con_firm:"
msgstr "Onayla"

#: pyanaconda/ui/gui/spokes/lib/passphrase.glade:273
msgid ""
"Warning: You won't be able to switch between keyboard layouts (from the "
"default one) when you decrypt your disks after install."
msgstr ""
"Uyarı: Diskler şifrelendikten sonra klavye düzenleri arasında "
"geçiş mümkün olmayacaktır."

#: pyanaconda/ui/gui/spokes/lib/refresh.glade:8
#: pyanaconda/ui/gui/spokes/lib/refresh.glade:71
msgid "RESCAN DISKS"
msgstr "DİSKLERİ YENİDEN TARA"

#: pyanaconda/ui/gui/spokes/lib/refresh.glade:25
msgctxt "GUI|Refresh Dialog|Rescan"
msgid "_Cancel"
msgstr "İptal"

#: pyanaconda/ui/gui/spokes/lib/refresh.glade:39
msgctxt "GUI|Refresh Dialog|Rescan"
msgid "_OK"
msgstr "TAMAM"

#: pyanaconda/ui/gui/spokes/lib/refresh.glade:87
msgid ""
"You can remove or insert additional disks at this time and press 'Rescan "
"Disks' below for the changes to take effect."
msgstr ""
"Bunu kaldırabilir veya şu anda başka diskler ekleyebilirsiniz. Değişikliklerin "
"etkili olması için aşağıdaki  'Diskleri Yeniden Tara'  tuşuna basabilirsiniz."

#: pyanaconda/ui/gui/spokes/lib/refresh.glade:118
msgid ""
"<b>Warning:</b> All storage changes made using the installer will be lost "
"when you press 'Rescan Disks'."
msgstr ""
"<b>Uyarı:</b> 'Diskleri Yeniden Tara' tuşuna bastığınızda yapılandırdığınız "
"tüm depolama değişiklikleri kaybolacak."

#: pyanaconda/ui/gui/spokes/lib/refresh.glade:143
msgctxt "GUI|Refresh Dialog|Rescan"
msgid "_Rescan Disks"
msgstr "Diskleri Yeniden Tara"

#: pyanaconda/ui/gui/spokes/lib/refresh.glade:176
#: pyanaconda/ui/gui/spokes/lib/refresh.glade:212
msgid "Scanning disks.  This may take a moment..."
msgstr "Diskler taranıyor.  Bu biraz zaman alabilir..."

#: pyanaconda/ui/gui/spokes/lib/refresh.glade:223
msgid ""
"You may <a href=\"\">go back to the main menu</a> to complete other\n"
"installation options while this scan completes."
msgstr ""
"Bu tarama tamamlanırken <a href=\"\">ana menüye geri dön</a> seçeneği ile\n"
"diğer yükleme seçeneklerini tanımlayabilirsiniz."

#: pyanaconda/ui/gui/spokes/lib/refresh.glade:254
msgid "Disk rescan complete."
msgstr "Tekrar disk tarama tamamlandı."

#: pyanaconda/ui/gui/spokes/lib/refresh.glade:269
msgid ""
"Pressing 'OK' below will take you to the disk selection screen where you "
"will need to re-select your disks."
msgstr ""
"'Tamam' a basılması sizi aşağıdaki Disk seçim ekranına götürecektir"
"Diskleri yeniden seçmeniz gerekir."

#: pyanaconda/ui/gui/spokes/lib/resize.glade:55
msgctxt "GUI|Reclaim Dialog"
msgid "_Cancel"
msgstr "İptal"

#: pyanaconda/ui/gui/spokes/lib/resize.glade:70
msgctxt "GUI|Reclaim Dialog"
msgid "_Reclaim space"
msgstr "Alanı Düzelt"

#: pyanaconda/ui/gui/spokes/lib/resize.glade:103
msgid "RECLAIM DISK SPACE"
msgstr "DİSK ALANINI DÜZELT"

#: pyanaconda/ui/gui/spokes/lib/resize.glade:120
msgid "Description goes here."
msgstr "Açıklama Buraya"

#: pyanaconda/ui/gui/spokes/lib/resize.glade:174
msgid "File System"
msgstr "Dosya Sistemi"

#: pyanaconda/ui/gui/spokes/lib/resize.glade:185
msgid "Reclaimable Space"
msgstr "Düzenlenebilir Alan"

#: pyanaconda/ui/gui/spokes/lib/resize.glade:196
#: pyanaconda/ui/gui/spokes/lib/summary.glade:144
msgid "Action"
msgstr "Eylem"

#: pyanaconda/ui/gui/spokes/lib/resize.glade:221
msgctxt "GUI|Reclaim Dialog"
msgid "_Preserve"
msgstr "Koru"

#: pyanaconda/ui/gui/spokes/lib/resize.glade:237
msgctxt "GUI|Reclaim Dialog"
msgid "_Delete"
msgstr "Sil"

#: pyanaconda/ui/gui/spokes/lib/resize.glade:253
msgctxt "GUI|Reclaim Dialog"
msgid "_Shrink"
msgstr "Daralt"

#: pyanaconda/ui/gui/spokes/lib/resize.glade:367
msgid "Reclaim"
msgstr "Düzelt"

#: pyanaconda/ui/gui/spokes/lib/summary.glade:25
#: pyanaconda/ui/gui/spokes/lib/summary.glade:89
msgid "SUMMARY OF CHANGES"
msgstr "DEĞİŞİM ÖZETİ"

#: pyanaconda/ui/gui/spokes/lib/summary.glade:42
msgctxt "GUI|Summary Dialog"
msgid "_Cancel & Return to Custom Partitioning"
msgstr "İptal Et ve Özel Bölümleme'ye Dön"

#: pyanaconda/ui/gui/spokes/lib/summary.glade:57
msgctxt "GUI|Summary Dialog"
msgid "_Accept Changes"
msgstr "Değişiklikleri Onayla"

#: pyanaconda/ui/gui/spokes/lib/summary.glade:105
msgid ""
"Your customizations will result in the following changes taking effect after "
"you return to the main menu and begin installation:"
msgstr ""
"Ana menüye dönerseniz veya kurulumu başlatırsanız özelleştirmeler "
"kabul edilir ve aşağıdaki değişikliklere neden olur:"

#: pyanaconda/ui/gui/spokes/lib/summary.glade:133
msgid "Order"
msgstr "Sıra"

#: pyanaconda/ui/gui/spokes/lib/summary.glade:166
msgid "Device"
msgstr "Aygıt"

#: pyanaconda/ui/gui/spokes/lib/summary.glade:177
msgid "Mount point"
msgstr "Bağlama noktası"

#: pyanaconda/ui/gui/spokes/lib/entropy_dialog.glade:35
msgid ""
"The system needs better quality of random data, you can improve it by typing "
"randomly on keyboard and moving your mouse. The installation will continue "
"automatically regardless of random data quality when time runs out."
msgstr ""
"Sistemin daha kaliteli işlem yapabilmesi için rastgele verilere ihtiyacı var, " 
"Klavyede rastgele yazarak ve farenizi hareket ettirerek kaliteyi artırabilirsiniz. "
"Otomatik olarak zaman biterse buna bakılmaksızın rasgele veri kalitesi devam ettirilecektir."

#: pyanaconda/ui/gui/spokes/lib/entropy_dialog.glade:53
msgid "Random data quality:"
msgstr "Rastgele veri kalitesi:"

#: pyanaconda/ui/gui/main.glade:24
msgctxt "GUI|Error Dialog"
msgid "_Exit Installer"
msgstr "Yükleyici'den Çık"

#: pyanaconda/ui/gui/main.glade:94
msgctxt "GUI|Main"
msgid "_No"
msgstr "Hayır"

#: pyanaconda/ui/gui/main.glade:108
msgctxt "GUI|Main"
msgid "_Yes"
msgstr "Evet"

#: pyanaconda/ui/gui/main.glade:151
msgid ""
"Are you sure you wish to quit the\n"
"installation process?"
msgstr ""
"Kurulumdan çıkmak istediğinize"
"eminmisiniz?"

#: pyanaconda/ui/gui/main.glade:175
msgid "Quit"
msgstr "Çık"

#: pyanaconda/ui/gui/hubs/summary.glade:8
#: pyanaconda/ui/gui/hubs/summary.glade:145
msgid "INSTALLATION SUMMARY"
msgstr "KURULUM ÖZETİ"

#: pyanaconda/ui/gui/hubs/summary.glade:81
msgctxt "GUI|Summary"
msgid "_Quit"
msgstr "Çık"

#: pyanaconda/ui/gui/hubs/summary.glade:97
msgctxt "GUI|Summary"
msgid "_Begin Installation"
msgstr "Kurulumu Başlat"

#: pyanaconda/ui/gui/hubs/summary.glade:128
msgid "We won't touch your disks until you click 'Begin Installation'."
msgstr "Siz 'Kurulumu Başlat' a tıklamadan disklere dokunulmaz."

#: pyanaconda/ui/gui/hubs/progress.glade:8
#: pyanaconda/ui/gui/hubs/progress.glade:260
msgid "CONFIGURATION"
msgstr "YAPILANDIRMA"

#: pyanaconda/ui/gui/hubs/progress.glade:88
msgid "Preparing to install"
msgstr "Yüklemek için hazırlanıyor"

#: pyanaconda/ui/gui/hubs/progress.glade:146
msgctxt "GUI|Progress"
msgid "_Finish configuration"
msgstr "Yapılandırma bitti"

#: pyanaconda/ui/gui/hubs/progress.glade:200
msgctxt "GUI|Progress"
msgid "_Reboot"
msgstr "Yeniden Başlat"

#: data/liveinst/liveinst.desktop.in.h:1
msgid "Install to Hard Drive"
msgstr "Sabit Diske Kur"

#: data/liveinst/liveinst.desktop.in.h:2
msgid "Install"
msgstr "Kur"

#: data/liveinst/liveinst.desktop.in.h:3
msgid "Install the live CD to your hard disk"
msgstr "Live CD'yi sabit diskinize yükleyin"

#: /home/bcl/Red_Hat/projs/anaconda/data/liveinst/console.apps/liveinst:5
msgid "Starting Install to Hard Drive"
msgstr "Sabit Diske Kurulum Başlıyor"

#: data/liveinst/gnome/fux-welcome.desktop.in.h:1
#: data/liveinst/gnome/fux-welcome:65
msgid "Hoşgeldiniz"
msgstr ""

#: data/liveinst/gnome/fux-welcome:94
msgid "Fux'u Dene"
msgstr ""

#: data/liveinst/gnome/fux-welcome:110
msgid ""
"Şu anda Fux'u canlı medya'dan çalıştırıyorsunuz.\n\n" 
"Fux'u şimdi veya herhangi bir zamanda \n\n"
"\"Sabit Diske Kur\" \n\n"
"Seçeneği ile yükleyebilirsiniz."
msgstr ""

#: data/liveinst/gnome/fux-welcome:128
msgid ""
"Fux'u kullanırken; Etkinlikler menüsünde'ki \n"
"\"Sabit Diske Kur\" \n"
"Kısayolu ile istediğiniz an yükleyebilirsiniz."
msgstr ""

#: data/liveinst/gnome/fux-welcome:131
msgid "Kapat"
msgstr ""
